<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="dns-prefetch" href="//www.google.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.googletagmanager.com" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2023.06.16.545386v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2023.06.16.545386v2.full.txt" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2023/06/26/2023.06.16.545386.atom" />
<meta name="HW.pisa" content="biorxiv;2023.06.16.545386v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis" />
<meta name="DC.Identifier" content="10.1101/2023.06.16.545386" />
<meta name="DC.Date" content="2023-06-26" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2023, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="A hallmark of age-associated neurodegenerative diseases is the aggregation of proteins. Aggregation of the protein tau defines tauopathies, which include Alzheimer’s disease and frontotemporal dementia. Specific neuronal subtypes are selectively vulnerable to the accumulation of tau aggregates, and subsequent dysfunction and death. The mechanisms underlying cell type-selective vulnerability are unknown. To systematically uncover the cellular factors controlling the accumulation of tau aggregates in human neurons, we conducted a genome-wide CRISPRi-based modifier screen in iPSC-derived neurons. The screen uncovered expected pathways, including autophagy, but also unexpected pathways including UFMylation and GPI anchor synthesis, that control tau oligomer levels. We identify the E3 ubiquitin ligase CUL5 as a tau interactor and potent modifier of tau levels. In addition, disruption of mitochondrial function increases tau oligomer levels and promotes proteasomal misprocessing of tau. These results reveal new principles of tau proteostasis in human neurons and pinpoint potential therapeutic targets for tauopathies.

### Competing Interest Statement

M. K. is an inventor on US Patent 11,254,933 related to CRISPRi and CRISPRa screening, serves on the Scientific Advisory Boards of Engine Biosciences, Casma Therapeutics, Cajal Neuroscience, Alector, and Montara Therapeutics, and is an advisor to Modulo Bio and Recursion Therapeutics. The other authors declare no competing interests." />
<meta name="DC.Contributor" content="Avi J. Samelson" />
<meta name="DC.Contributor" content="Nabeela Ariqat" />
<meta name="DC.Contributor" content="Justin McKetney" />
<meta name="DC.Contributor" content="Gita Rohanitazangi" />
<meta name="DC.Contributor" content="Celeste Parra Bravo" />
<meta name="DC.Contributor" content="Darrin Goodness" />
<meta name="DC.Contributor" content="Ruilin Tian" />
<meta name="DC.Contributor" content="Parker Grosjean" />
<meta name="DC.Contributor" content="Romany Abskharon" />
<meta name="DC.Contributor" content="David Eisenberg" />
<meta name="DC.Contributor" content="Nicholas M. Kanaan" />
<meta name="DC.Contributor" content="Li Gan" />
<meta name="DC.Contributor" content="Carlo Condello" />
<meta name="DC.Contributor" content="Danielle L. Swaney" />
<meta name="DC.Contributor" content="Martin Kampmann" />
<meta name="article:published_time" content="2023-06-26" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Summary&lt;/h3&gt;
&lt;p&gt;A hallmark of age-associated neurodegenerative diseases is the aggregation of proteins. Aggregation of the protein tau defines tauopathies, which include Alzheimer’s disease and frontotemporal dementia. Specific neuronal subtypes are selectively vulnerable to the accumulation of tau aggregates, and subsequent dysfunction and death. The mechanisms underlying cell type-selective vulnerability are unknown. To systematically uncover the cellular factors controlling the accumulation of tau aggregates in human neurons, we conducted a genome-wide CRISPRi-based modifier screen in iPSC-derived neurons. The screen uncovered expected pathways, including autophagy, but also unexpected pathways including UFMylation and GPI anchor synthesis, that control tau oligomer levels. We identify the E3 ubiquitin ligase CUL5 as a tau interactor and potent modifier of tau levels. In addition, disruption of mitochondrial function increases tau oligomer levels and promotes proteasomal misprocessing of tau. These results reveal new principles of tau proteostasis in human neurons and pinpoint potential therapeutic targets for tauopathies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2023/01/01" />
<meta name="citation_mjid" content="biorxiv;2023.06.16.545386v2" />
<meta name="citation_id" content="2023.06.16.545386v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2023.06.16.545386v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2023.06.16.545386v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2023.06.16.545386v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386.full.pdf" />
<meta name="citation_doi" content="10.1101/2023.06.16.545386" />
<meta name="citation_num_pages" content="43" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2023.06.16.545386" />
<meta name="citation_author" content="Avi J. Samelson" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author" content="Nabeela Ariqat" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author" content="Justin McKetney" />
<meta name="citation_author_institution" content="University of California San Francisco, Quantitative Biosciences Institute (QBI)" />
<meta name="citation_author_institution" content="University of California San Francisco, Department of Cellular and Molecular Pharmacology" />
<meta name="citation_author_institution" content="Gladstone Institute of Data Science and Biotechnology, Gladstone Institutes" />
<meta name="citation_author" content="Gita Rohanitazangi" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author" content="Celeste Parra Bravo" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author_institution" content="Helen and Robert Appel Alzheimer Disease Research Institute, Feil Family Brain and Mind Research Institute, Weill Cornell Medicine" />
<meta name="citation_author" content="Darrin Goodness" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author" content="Ruilin Tian" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author" content="Parker Grosjean" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author" content="Romany Abskharon" />
<meta name="citation_author_institution" content="Departments of Chemistry and Biochemistry and Biological Chemistry, UCLA-DOE Institute, UCLA" />
<meta name="citation_author" content="David Eisenberg" />
<meta name="citation_author_institution" content="Departments of Chemistry and Biochemistry and Biological Chemistry, UCLA-DOE Institute, UCLA" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute UCLA" />
<meta name="citation_author" content="Nicholas M. Kanaan" />
<meta name="citation_author_institution" content="Department of Translational Neuroscience, Michigan State University" />
<meta name="citation_author" content="Li Gan" />
<meta name="citation_author_institution" content="Helen and Robert Appel Alzheimer Disease Research Institute, Feil Family Brain and Mind Research Institute, Weill Cornell Medicine" />
<meta name="citation_author" content="Carlo Condello" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author_institution" content="Department of Neurology, University of California San Francisco" />
<meta name="citation_author" content="Danielle L. Swaney" />
<meta name="citation_author_institution" content="University of California San Francisco, Quantitative Biosciences Institute (QBI)" />
<meta name="citation_author_institution" content="University of California San Francisco, Department of Cellular and Molecular Pharmacology" />
<meta name="citation_author_institution" content="Gladstone Institute of Data Science and Biotechnology, Gladstone Institutes" />
<meta name="citation_author" content="Martin Kampmann" />
<meta name="citation_author_institution" content="Institute for Neurodegenerative Diseases, University of California" />
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of California San Francisco" />
<meta name="citation_author_email" content="martin.kampmann@ucsf.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3819-7019" />
<meta name="citation_reference" content="Chang, C.W., Shao, E., and Mucke, L. (2021). Tau: Enabler of diverse brain disorders and target of rapidly evolving therapeutic strategies. Science 371. 10.1126/science.abb8255." />
<meta name="citation_reference" content="Wang, Y., and Mandelkow, E. (2016). Tau in physiology and pathology. Nature reviews. Neuroscience 17, 5–21. 10.1038/nrn.2015.1." />
<meta name="citation_reference" content="Tracy, T.E., and Gan, L. (2018). Tau-mediated synaptic and neuronal dysfunction in neurodegenerative disease. Curr Opin Neurobiol 51, 134–138. 10.1016/j.conb.2018.04.027." />
<meta name="citation_reference" content="Guo, T., Noble, W., and Hanger, D.P. (2017). Roles of tau protein in health and disease. Acta Neuropathol 133, 665–704. 10.1007/s00401-017-1707-9." />
<meta name="citation_reference" content="Lee, V.M., Goedert, M., and Trojanowski, J.Q. (2001). Neurodegenerative tauopathies. Annu Rev Neurosci 24, 1121–1159. 10.1146/annurev.neuro.24.1.1121." />
<meta name="citation_reference" content="Hutton, M., Lendon, C.L., Rizzu, P., Baker, M., Froelich, S., Houlden, H., Pickering-Brown, S., Chakraverty, S., Isaacs, A., Grover, A., et al. (1998). Association of missense and 5’-splice-site mutations in tau with the inherited dementia FTDP-17. Nature 393, 702–705. 10.1038/31508." />
<meta name="citation_reference" content="Poorkaj, P., Grossman, M., Steinbart, E., Payami, H., Sadovnick, A., Nochlin, D., Tabira, T., Trojanowski, J.Q., Borson, S., Galasko, D., et al. (2001). Frequency of tau gene mutations in familial and sporadic cases of non-Alzheimer dementia. Arch Neurol 58, 383–387. 10.1001/archneur.58.3.383." />
<meta name="citation_reference" content="van Swieten, J., and Spillantini, M.G. (2007). Hereditary frontotemporal dementia caused by Tau gene mutations. Brain Pathol 17, 63–73. 10.1111/j.1750-3639.2007.00052.x." />
<meta name="citation_reference" content="Goedert, M., and Spillantini, M.G. (2011). Pathogenesis of the tauopathies. Journal of Molecular Neuroscience 45, 425–431. 10.1007/s12031-011-9593-4." />
<meta name="citation_reference" content="Braak, H., and Braak, E. (1991). Neuropathological stageing of Alzheimer-related changes. Acta Neuropathol 82, 239–259. 10.1007/BF00308809." />
<meta name="citation_reference" content="Leng, K., Li, E., Eser, R., Piergies, A., Sit, R., Tan, M., Neff, N., Li, S.H., Rodriguez, R.D., Suemoto, C.K., et al. (2021). Molecular characterization of selectively vulnerable neurons in Alzheimer’s disease. Nat Neurosci 24, 276–287. 10.1038/s41593-020-00764-7." />
<meta name="citation_reference" content="Schö, M., Lockhart, S.N., Schonhaut, D.R., Schwimmer, H.D., Rabinovici, G.D., Correspondence, W.J.J., Schö Ll, M., O ’neil, J.P., Janabi, M., Ossenkoppele, R., et al. (2016). PET Imaging of Tau Deposition in the Aging Human Brain. 971–982. 10.1016/j.neuron.2016.01.028." />
<meta name="citation_reference" content="Seeley, W.W., Crawford, R.K., Zhou, J., Miller, B.L., and Greicius, M.D. (2009). Neurodegenerative diseases target large-scale human brain networks. Neuron 62, 42–52. 10.1016/j.neuron.2009.03.024." />
<meta name="citation_reference" content="Shi, Y., Zhang, W., Yang, Y., Murzin, A.G., Falcon, B., Kotecha, A., van Beers, M., Tarutani, A., Kametani, F., Garringer, H.J., et al. (2021). Structure-based classification of tauopathies. Nature 598, 359–363. 10.1038/s41586-021-03911-7." />
<meta name="citation_reference" content="Arakhamia, T., Lee, C.E., Carlomagno, Y., Seyfried, N.T., Petrucelli, L., Fitzpatrick, A.W.P., Arakhamia, T., Lee, C.E., Carlomagno, Y., Duong, D.M., et al. (2020). Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains. Cell 180, 1–12. 10.1016/j.cell.2020.01.027." />
<meta name="citation_reference" content="Wesseling, H., Mair, W., Kumar, M., Schlaffner, C.N., Tang, S., Beerepoot, P., Fatou, B., Guise, A.J., Cheng, L., Takeda, S., et al. (2020). Tau PTM Profiles Identify Patient Heterogeneity and Stages of Alzheimer’s Disease. Cell 183, 1699–1713.e13. 10.1016/j.cell.2020.10.029." />
<meta name="citation_reference" content="Bellenguez, C., Küçükali, F., Jansen, I.E., Kleineidam, L., Moreno-Grau, S., Amin, N., Naj, A.C., Campos-Martin, R., Grenier-Boley, B., Andrade, V., et al. (2022). New insights into the genetic etiology of Alzheimer’s disease and related dementias. Nat Genet 54, 412–436. 10.1038/s41588-022-01024-z." />
<meta name="citation_reference" content="Wightman, D.P., Jansen, I.E., Savage, J.E., Shadrin, A.A., Bahrami, S., Holland, D., Rongve, A., Børte, S., Winsvold, B.S., Drange, O.K., et al. (2021). A genome-wide association study with 1,126,563 individuals identifies new risk loci for Alzheimer’s disease. Nat Genet 53, 1276–1282. 10.1038/s41588-021-00921-z." />
<meta name="citation_reference" content="Jansen, I.E., Savage, J.E., Watanabe, K., Bryois, J., Williams, D.M., Steinberg, S., Sealock, J., Karlsson, I.K., Hägg, S., Athanasiu, L., et al. (2019). Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer’s disease risk. Nat Genet 51, 404–413. 10.1038/s41588-018-0311-9." />
<meta name="citation_reference" content="Andrews, S.J., Fulton-Howard, B., and Goate, A. (2020). Interpretation of risk loci from genome-wide association studies of Alzheimer’s disease. Lancet Neurol 19, 326–335. 10.1016/S1474-4422(19)30435-1." />
<meta name="citation_reference" content="Mathys, H., Davila-Velderrain, J., Peng, Z., Gao, F., Mohammadi, S., Young, J.Z., Menon, M., He, L., Abdurrob, F., Jiang, X., et al. (2019). Single-cell transcriptomic analysis of Alzheimer’s disease. Nature 570, 332–337. 10.1038/s41586-019-1195-2." />
<meta name="citation_reference" content="Otero-Garcia, M., Mahajani, S.U., Wakhloo, D., Tang, W., Xue, Y.-Q., Morabito, S., Pan, J., Oberhauser, J., Madira, A.E., Shakouri, T., et al. (2022). Molecular signatures underlying neurofibrillary tangle susceptibility in Alzheimer’s disease. Neuron 110, 2929–2948.e8. 10.1016/j.neuron.2022.06.021." />
<meta name="citation_reference" content="Tian, R., Gachechiladze, M.A., Ludwig, C.H., Laurie, M.T., Hong, J.Y., Nathaniel, D., Prabhu, A.V., Fernandopulle, M.S., Patel, R., Abshari, M., et al. (2019). CRISPR Interference-Based Platform for Multimodal Genetic Screens in Human iPSC-Derived Neurons. Neuron 104, 239–255.e12. 10.1016/j.neuron.2019.07.014." />
<meta name="citation_reference" content="Hong, M. (1998). Mutation-Specific Functional Impairments in Distinct Tau Isoforms of Hereditary FTDP-17. Science 282, 1914–1917. 10.1126/science.282.5395.1914." />
<meta name="citation_reference" content="Spina, S., Schonhaut, D.R., Boeve, B.F., Seeley, W.W., Ossenkoppele, R., O’Neil, J.P., Lazaris, A., Rosen, H.J., Boxer, A.L., Perry, D.C., et al. (2017). Frontotemporal dementia with the V337M MAPT mutation: Tau-PET and pathology correlations. Neurology 88, 758– 766. 10.1212/WNL.0000000000003636." />
<meta name="citation_reference" content="Chung, C.W., Song, Y.H., Kim, I.K., Yoon, W.J., Ryu, B.R., Jo, D.G., Woo, H.N., Kwon, Y.K., Kim, H.H., Gwag, B.J., et al. (2001). Proapoptotic effects of tau cleavage product generated by caspase-3. Neurobiol Dis 8, 162–172. 10.1006/nbdi.2000.0335." />
<meta name="citation_reference" content="Flach, K., Hilbrich, I., Schiffmann, A., Gärtner, U., Krüger, M., Leonhardt, M., Waschipky, H., Wick, L., Arendt, T., and Holzer, M. (2012). Tau oligomers impair artificial membrane integrity and cellular viability. J Biol Chem 287, 43223–43233. 10.1074/jbc.M112.396176." />
<meta name="citation_reference" content="Jackson, G.R., Lasagna-reeves, C.A., Castillo-carranza, D.L., Guerrero-mu, M.J., and Kayed, R. (2010). Preparation and Characterization of Neurotoxic Tau Oligomers †. 10039–10041. 10.1021/bi1016233." />
<meta name="citation_reference" content="Lasagna-Reeves, C.A., Castillo-Carranza, D.L., Sengupta, U., Sarmiento, J., Troncoso, J., Jackson, G.R., and Kayed, R. (2012). Identification of oligomers at early stages of tau aggregation in Alzheimer’s disease. The FASEB Journal 26, 1946–1959. 10.1096/fj.11-199851." />
<meta name="citation_reference" content="Lasagna-Reeves, C.A., Castillo-Carranza, D.L., Sengupta, U., Guerrero-Munoz, M.J., Kiritoshi, T., Neugebauer, V., Jackson, G.R., and Kayed, R. (2012). Alzheimer brain-derived tau oligomers propagate pathology from endogenous tau. Scientific Reports 2. 10.1038/srep00700." />
<meta name="citation_reference" content="Horlbeck, M.A., Gilbert, L.A., Villalta, J.E., Adamson, B., Pak, R.A., Chen, Y., Fields, A.P., Park, C.Y., Corn, J.E., Kampmann, M., et al. (2016). Compact and highly active next-generation libraries for CRISPR-mediated gene repression and activation. eLife 5, 1–20. 10.7554/eLife.19760." />
<meta name="citation_reference" content="Caballero, B., Bourdenx, M., Luengo, E., Diaz, A., Sohn, P.D., Chen, X., Wang, C., Juste, Y.R., Wegmann, S., Patel, B., et al. (2021). Acetylated tau inhibits chaperone-mediated autophagy and promotes tau pathology propagation in mice. Nat Commun 12, 2238. 10.1038/s41467-021-22501-9." />
<meta name="citation_reference" content="Djajadikerta, A., Keshri, S., Pavel, M., Prestil, R., Ryan, L., and Rubinsztein, D.C. (2020). Autophagy Induction as a Therapeutic Strategy for Neurodegenerative Diseases. J Mol Biol 432, 2799–2821. 10.1016/j.jmb.2019.12.035." />
<meta name="citation_reference" content="Menzies, F.M., Fleming, A., Caricasole, A., Bento, C.F., Andrews, S.P., Ashkenazi, A., Füllgrabe, J., Jackson, A., Jimenez Sanchez, M., Karabiyik, C., et al. (2017). Autophagy and Neurodegeneration: Pathogenic Mechanisms and Therapeutic Opportunities. Neuron 93, 1015–1034. 10.1016/j.neuron.2017.01.022." />
<meta name="citation_reference" content="Silva, M.C., Nandi, G.A., Tentarelli, S., Gurrell, I.K., Jamier, T., Lucente, D., Dickerson, B.C., Brown, D.G., Brandon, N.J., and Haggarty, S.J. (2020). Prolonged tau clearance and stress vulnerability rescue by pharmacological activation of autophagy in tauopathy neurons. Nat Commun 11, 3258. 10.1038/s41467-020-16984-1." />
<meta name="citation_reference" content="Jiang, L., Lin, W., Zhang, C., Ash, P.E.A., Verma, M., Kwan, J., van Vliet, E., Yang, Z., Cruz, A.L., Boudeau, S., et al. (2021). Interaction of tau with HNRNPA2B1 and N6-methyladenosine RNA mediates the progression of tauopathy. Molecular Cell 81, 4209–4227.e12. 10.1016/j.molcel.2021.07.038." />
<meta name="citation_reference" content="Celeste Parra Bravo, Alice Maria Giani, Jesus Madero-Perez, Zeping Zhao, Avi J Samelson, Man Ying Wong, Alessandro Evangelisti, Li Fan, Tatyana Pozner, Maria Mercedes, et al. (2023). Human iPSC 4R tauopathy model uncovers modifiers of tau propagation. bioRxiv, 2023.06.19.544278. 10.1101/2023.06.19.544278." />
<meta name="citation_reference" content="Kanaan, N.M., Cox, K., Alvarez, V.E., Stein, T.D., Poncil, S., and McKee, A.C. (2016). Characterization of Early Pathological Tau Conformations and Phosphorylation in Chronic Traumatic Encephalopathy. J Neuropathol Exp Neurol 75, 19–34. 10.1093/jnen/nlv001." />
<meta name="citation_reference" content="Abskharon, R., Seidler, P.M., Sawaya, M.R., Cascio, D., Yang, T.P., Philipp, S., Williams, C.K., Newell, K.L., Ghetti, B., DeTure, M.A., et al. (2020). Crystal structure of a conformational antibody that binds tau oligomers and inhibits pathological seeding by extracts from donors with Alzheimer’s disease. The Journal of biological chemistry, 1–28. 10.1074/jbc.RA120.013638." />
<meta name="citation_reference" content="Bano, I., Soomro, A.S., Abbas, S.Q., Ahmadi, A., Hassan, S.S.U., Behl, T., and Bungau, S. (2022). A Comprehensive Review of Biological Roles and Interactions of Cullin-5 Protein. ACS Omega 7, 5615–5624. 10.1021/acsomega.1c06890." />
<meta name="citation_reference" content="Gao, F., Fan, Y., Zhou, B., Guo, W., Jiang, X., Shi, J., and Ren, C. (2020). The functions and properties of cullin-5, a potential therapeutic target for cancers. Am J Transl Res 12, 618– 632." />
<meta name="citation_reference" content="Yu, X., Yu, Y., Liu, B., Luo, K., Kong, W., Mao, P., and Yu, X.-F. (2003). Induction of APOBEC3G ubiquitination and degradation by an HIV-1 Vif-Cul5-SCF complex. Science 302, 1056–1060. 10.1126/science.1089591." />
<meta name="citation_reference" content="Tian, R., Abarientos, A., Hong, J., Hashemi, S.H., Yan, R., Dräger, N., Leng, K., Nalls, M.A., Singleton, A.B., Xu, K., et al. (2021). Genome-wide CRISPRi/a screens in human neurons link lysosomal failure to ferroptosis. Nature Neuroscience 24, 1020–1034. 10.1038/s41593-021-00862-0." />
<meta name="citation_reference" content="David, D.C., Layfield, R., Serpell, L., Narain, Y., Goedert, M., and Spillantini, M.G. (2002). Proteasomal degradation of tau protein. J Neurochem 83, 176–185. 10.1046/j.1471-4159.2002.01137.x." />
<meta name="citation_reference" content="Quinn, J.P., Corbett, N.J., Kellett, K.A.B., and Hooper, N.M. (2018). Tau Proteolysis in the Pathogenesis of Tauopathies: Neurotoxic Fragments and Novel Biomarkers. Journal of Alzheimer’s disease : JAD 63, 13–33. 10.3233/JAD-170959." />
<meta name="citation_reference" content="Sampognaro, P.J., Arya, S., Knudsen, G.M., Gunderson, E.L., Sandoval-Perez, A., Hodul, M., Bowles, K., Craik, C.S., Jacobson, M.P., and Kao, A.W. (2023). Mutations in α-synuclein, TDP-43 and tau prolong protein half-life through diminished degradation by lysosomal proteases. Mol Neurodegener 18, 29. 10.1186/s13024-023-00621-8." />
<meta name="citation_reference" content="Ukmar-Godec, T., Fang, P., Ibáñez de Opakua, A., Henneberg, F., Godec, A., Pan, K.T., Cima-Omori, M.S., Chari, A., Mandelkow, E., Urlaub, H., et al. (2020). Proteasomal degradation of the intrinsically disordered protein tau at single-residue resolution. Science Advances 6, 1–13. 10.1126/sciadv.aba3916." />
<meta name="citation_reference" content="Kors, S., Geijtenbeek, K., Reits, E., and Schipper-Krom, S. (2019). Regulation of Proteasome Activity by (Post-)transcriptional Mechanisms. Front Mol Biosci 6, 48. 10.3389/fmolb.2019.00048." />
<meta name="citation_reference" content="Rechsteiner, M., and Hill, C.P. (2005). Mobilizing the proteolytic machine: cell biological roles of proteasome activators and inhibitors. Trends Cell Biol 15, 27–33. 10.1016/j.tcb.2004.11.003." />
<meta name="citation_reference" content="Barthélemy, N.R., Gabelle, A., Hirtz, C., Fenaille, F., Sergeant, N., Schraen-Maschke, S., Vialaret, J., Buée, L., Junot, C., Becher, F., et al. (2016). Differential Mass Spectrometry Profiles of Tau Protein in the Cerebrospinal Fluid of Patients with Alzheimer’s Disease, Progressive Supranuclear Palsy, and Dementia with Lewy Bodies. J Alzheimers Dis 51, 1033–1043. 10.3233/JAD-150962." />
<meta name="citation_reference" content="Cicognola, C., Brinkmalm, G., Wahlgren, J., Portelius, E., Gobom, J., Cullen, N.C., Hansson, O., Parnetti, L., Constantinescu, R., Wildsmith, K., et al. (2019). Novel tau fragments in cerebrospinal fluid: relation to tangle pathology and cognitive decline in Alzheimer’s disease. Acta Neuropathol 137, 279–296. 10.1007/s00401-018-1948-2." />
<meta name="citation_reference" content="Sato, C., Barthélemy, N.R., Mawuenyega, K.G., Patterson, B.W., Gordon, B.A., Jockel-Balsarotti, J., Sullivan, M., Crisp, M.J., Kasten, T., Kirmess, K.M., et al. (2018). Tau Kinetics in Neurons and the Human Central Nervous System. Neuron 97, 1284–1298.e7. 10.1016/j.neuron.2018.02.015." />
<meta name="citation_reference" content="Duan, L., Hu, M., Tamm, J.A., Grinberg, Y.Y., Shen, F., Chai, Y., Xi, H., Gibilisco, L., Ravikumar, B., Gautam, V., et al. (2021). Arrayed CRISPR reveals genetic regulators of tau aggregation, autophagy and mitochondria in Alzheimer’s disease model. Sci Rep 11, 2879. 10.1038/s41598-021-82658-7." />
<meta name="citation_reference" content="Kim, J., de Haro, M., Al-Ramahi, I., Garaicoechea, L.L., Jeong, H.-H., Sonn, J.Y., Tadros, B., Liu, Z., Botas, J., and Zoghbi, H.Y. (2023). Evolutionarily conserved regulators of tau identify targets for new therapies. Neuron 111, 824–838.e7. 10.1016/j.neuron.2022.12.012." />
<meta name="citation_reference" content="Sanchez, C.G., Acker, C.M., Gray, A., Varadarajan, M., Song, C., Cochran, N.R., Paula, S., Lindeman, A., An, S., McAllister, G., et al. (2021). Genome-wide CRISPR screen identifies protein pathways modulating tau protein levels in neurons. Communications Biology 4, 1–14. 10.1038/s42003-021-02272-1." />
<meta name="citation_reference" content="Yan, Y., Wang, X., Chaput, D., Shin, M.-K., Koh, Y., Gan, L., Pieper, A.A., Woo, J.-A.A., and Kang, D.E. (2022). X-linked ubiquitin-specific peptidase 11 increases tauopathy vulnerability in women. Cell 185, 3913–3930.e19. 10.1016/j.cell.2022.09.002." />
<meta name="citation_reference" content="Chaves, R.S., Melo, T.Q., Martins, S.A., and Ferrari, M.F.R. (2010). Protein aggregation containing β-amyloid, α-synuclein and hyperphosphorylated τ in cultured cells of hippocampus, substantia nigra and locus coeruleus after rotenone exposure. BMC Neurosci 11, 144. 10.1186/1471-2202-11-144." />
<meta name="citation_reference" content="Höglinger, G.U., Lannuzel, A., Khondiker, M.E., Michel, P.P., Duyckaerts, C., Féger, J., Champy, P., Prigent, A., Medja, F., Lombes, A., et al. (2005). The mitochondrial complex I inhibitor rotenone triggers a cerebral tauopathy. Journal of Neurochemistry 95, 930–939. 10.1111/j.1471-4159.2005.03493.x." />
<meta name="citation_reference" content="Chirichigno, J.W., Manfredi, G., Beal, M.F., and Albers, D.S. (2002). Stress-induced mitochondrial depolarization and oxidative damage in PSP cybrids. Brain Res 951, 31–35. 10.1016/s0006-8993(02)03101-3." />
<meta name="citation_reference" content="Tracy, T.E., Madero-Pérez, J., Swaney, D.L., Chang, T.S., Moritz, M., Konrad, C., Ward, M.E., Stevenson, E., Hüttenhain, R., Kauwe, G., et al. (2022). Tau interactome maps synaptic and mitochondrial processes associated with neurodegeneration. Cell, 1–360. 10.1016/j.cell.2021.12.041." />
<meta name="citation_reference" content="Sohn, P.D., Huang, C.T., Yan, R., Xu, K., Kosik, K.S., Gan, L., Sohn, P.D., Huang, C.T., Yan, R., Fan, L., et al. (2019). Pathogenic Tau Impairs Axon Initial Segment Plasticity and Excitability Homeostasis Article Pathogenic Tau Impairs Axon Initial Segment Plasticity and Excitability Homeostasis. 1–13. 10.1016/j.neuron.2019.08.008." />
<meta name="citation_reference" content="Wang, C., Ward, M.E., Chen, R., Liu, K., Tracy, T.E., Chen, X., Xie, M., Sohn, P.D., Ludwig, C., Meyer-Franke, A., et al. (2017). Scalable Production of iPSC-Derived Human Neurons to Identify Tau-Lowering Compounds by High-Content Screening. Stem Cell Reports 9, 1221–1233. 10.1016/j.stemcr.2017.08.019." />
<meta name="citation_reference" content="Gilbert, L.A., Horlbeck, M.A., Adamson, B., Villalta, J.E., Chen, Y., Whitehead, E.H., Guimaraes, C., Panning, B., Ploegh, H.L., Bassik, M.C., et al. (2014). Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation. Cell 159, 647–661. 10.1016/j.cell.2014.09.029." />
<meta name="citation_reference" content="Chen, E.Y., Tan, C.M., Kou, Y., Duan, Q., Wang, Z., Meirelles, G., Clark, N.R., and Ma’ayan, A. (2013). Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC Bioinformatics 14, 128. 10.1186/1471-2105-14-128." />
<meta name="citation_reference" content="Kuleshov, M.V., Jones, M.R., Rouillard, A.D., Fernandez, N.F., Duan, Q., Wang, Z., Koplev, S., Jenkins, S.L., Jagodnik, K.M., Lachmann, A., et al. (2016). Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res 44, W90–97. 10.1093/nar/gkw377." />
<meta name="citation_reference" content="Xie, Z., Bailey, A., Kuleshov, M.V., Clarke, D.J.B., Evangelista, J.E., Jenkins, S.L., Lachmann, A., Wojciechowicz, M.L., Kropiwnicki, E., Jagodnik, K.M., et al. (2021). Gene Set Knowledge Discovery with Enrichr. Curr Protoc 1, e90. 10.1002/cpz1.90." />
<meta name="twitter:title" content="CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="A hallmark of age-associated neurodegenerative diseases is the aggregation of proteins. Aggregation of the protein tau defines tauopathies, which include Alzheimer’s disease and frontotemporal dementia. Specific neuronal subtypes are selectively vulnerable to the accumulation of tau aggregates, and subsequent dysfunction and death. The mechanisms underlying cell type-selective vulnerability are unknown. To systematically uncover the cellular factors controlling the accumulation of tau aggregates in human neurons, we conducted a genome-wide CRISPRi-based modifier screen in iPSC-derived neurons. The screen uncovered expected pathways, including autophagy, but also unexpected pathways including UFMylation and GPI anchor synthesis, that control tau oligomer levels. We identify the E3 ubiquitin ligase CUL5 as a tau interactor and potent modifier of tau levels. In addition, disruption of mitochondrial function increases tau oligomer levels and promotes proteasomal misprocessing of tau. These results reveal new principles of tau proteostasis in human neurons and pinpoint potential therapeutic targets for tauopathies.

### Competing Interest Statement

M. K. is an inventor on US Patent 11,254,933 related to CRISPRi and CRISPRa screening, serves on the Scientific Advisory Boards of Engine Biosciences, Casma Therapeutics, Cajal Neuroscience, Alector, and Montara Therapeutics, and is an advisor to Modulo Bio and Recursion Therapeutics. The other authors declare no competing interests." />
<meta name="og-title" property="og:title" content="CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2023.06.16.545386v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="A hallmark of age-associated neurodegenerative diseases is the aggregation of proteins. Aggregation of the protein tau defines tauopathies, which include Alzheimer’s disease and frontotemporal dementia. Specific neuronal subtypes are selectively vulnerable to the accumulation of tau aggregates, and subsequent dysfunction and death. The mechanisms underlying cell type-selective vulnerability are unknown. To systematically uncover the cellular factors controlling the accumulation of tau aggregates in human neurons, we conducted a genome-wide CRISPRi-based modifier screen in iPSC-derived neurons. The screen uncovered expected pathways, including autophagy, but also unexpected pathways including UFMylation and GPI anchor synthesis, that control tau oligomer levels. We identify the E3 ubiquitin ligase CUL5 as a tau interactor and potent modifier of tau levels. In addition, disruption of mitochondrial function increases tau oligomer levels and promotes proteasomal misprocessing of tau. These results reveal new principles of tau proteostasis in human neurons and pinpoint potential therapeutic targets for tauopathies.

### Competing Interest Statement

M. K. is an inventor on US Patent 11,254,933 related to CRISPRi and CRISPRa screening, serves on the Scientific Advisory Boards of Engine Biosciences, Casma Therapeutics, Cajal Neuroscience, Alector, and Montara Therapeutics, and is an advisor to Modulo Bio and Recursion Therapeutics. The other authors declare no competing interests." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2023-06-26" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2023.06.16.545386v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2023.06.16.545386v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/3222388" />
    <title>CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]> */
</style>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__pWyBeQHtoijrNnOoDz9ZPdiNPirDrSCPOz0Q1CDCeno__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__LebTxUj4C7IVVFbkkwQ2Sd2OYxP3y19pEQs79XX-0z4__igagZiGB2PktJMjUpZ2u3tnc56saoUGIrD5N0KSRmI4__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__HdtQnswJFIO79IxmNaYwmxjlQ7F2Q1P-IaJrZ-M5bOU__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.js"></script>
<script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&amp;render=explicit&amp;onload=drupalRecaptchaOnload"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__1l8Wa0iuIek7SEVmMuU0Y9TlAvRR-XZVfl1u9ezOPes__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" async="async" src="https://www.googletagmanager.com/gtag/js?id=G-RZD586MC3Q"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments)};gtag("js", new Date());gtag("set", "developer_id.dMDhkMT", true);gtag("config", "G-RZD586MC3Q", {"groups":"default","anonymize_ip":true});
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RNooLN2RN8NbvQ1WQ7vrbktoKG8slEYb8kIvasj3Gfo"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"3222388","apath":"\/biorxiv\/early\/2023\/06\/26\/2023.06.16.545386.atom","pisa":"biorxiv;2023.06.16.545386v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2023.06.16.545386v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2023.06.16.545386v2","title":"CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis","identifier":"node\/3222388"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U6XOhj\\\\\\\/Z+ZkiACFJBsbYxKNg6V59cC+ABBIDiCj\\\\\\\/RGk\\\\\\\/odoAhcikIUe184CMlxm1ioh3rcjyiTQhC+XNV+Ga5wXMGN50H0iTsGpjBSMcsYOvBHEK2Stw26VFIXgk4D3iPfNqUiXAS28SZ\\\\\\\/AHZ34oX3EdUv7LSUwVjconFYfqmFMv\\\\\\\/jmVCZzDhueL9sNpyo\\\\\\\/QF0xUPYOpHbcYva1Q\\\\\\\/qnaOsbOm5MgkxpLUMiESGoSyfeV+vMKcU8ZW5xznbnL0Uvf17AY4kwKJzbErt93Ft3bkVCrq8bO6qyPrpwINtAPmuZ00MnGku7J0yCEaa+N6K3fqpyI2WVWKb16lDmlueNu6pgutxTyAK4V5A3zQx4VL0a9Phkp+GUNKEiZt7o011440Oqhfz2c9shPg83EhDEi59nozBivzxtKe9lC5c1gKmRhoCl9OGFbS9sG53NXXulXOuIFpFKr4MLrq6wgDn0AQ8xZjeFOpSGlc9OXtXEJsjr3c1dsYrsz\\\\\\\/KiVMZgdfLQZM82QCkkuGbsxQ+UQa8vr8BF6Y4ZVDoTBAnNcBBnFplKOoom\\\\\\\/7jEu8eAFYbkP0SGRJxv9EK1LCHpS4xFSCPaVq65yuQuEf05\\\\\\\/oAdOHtlWxnif\\\\\\\/QgfUAXrKneZG\\\\\\\/tKl7rMdv0Fy3yrJg6Xuco2Yc2uNLH2IX0wVB7glH4\\\\\\\/+XfS9rQUwxuBAandO\\\\\\\/UaIyKTKFqcHayri47xiHDLi3AbUOviKAnALgfzbKrZr9ZIjTD\\\\\\\/OU\\\\\\\/6iEWDbqgvpiQk8asTs4wQ\\\\\\\/HtW0YRrBl1KJy0+rsz1GVC0WRh5nwlxcF\\\\\\\/mws2qdqcRVOgr9HlIOzFVSSLVcx552tLHt51jrKdMm2zersUtMg3bosQ1kuvNSU\\\\\\\/XA4f3OUL8UO5Ew6OvxTUaLs7SQ3RyV138VBxiy+wPV35aiVd2HAr2JVPadn8RbtH66C1PkAyG2E4njD3qj0DgVbG\\\\\\\/JxFrYUMoCnFk\\\\\\\/qgJ1VhmVN6eWYQGru2jz8vQMr8L9m\\\\\\\/engUSANZMbw6bDLU+8zuxzVHQiUmDXxuskjVPIWWion+R3Vs0EtOMiQTW0+Dg\\\\\\\/xTLI86r1VNfMy2vgEIchA9Or4NpC5PBYV6FnK8izb5svbkBV16\\\\\\\/xTKvJkFO+ERJEZd+xcGZo7B0694GZ1yDaD\\\\\\\/WFlTsurbPFx1fJqL2N3WgriwArsCy2+r56YE1ofPd0A50JYmaUf2gA9UI2u17a8RuFea209U3eo6Tu9F3xmP1mwZcWOnqG7lSb7ftkuPWi9WzKB3aEVhWoThXKqjZ2jaT+OZ1schPnr617Ou9kSxkLRNadP+5ubrW5vnr33r5KIWYjeqeafEoE6GzgsMi+VCISX1DljJsIw3eJEY+l94Fax1m\\\\\\\/ng\\\\\\\/Urp1bjQjvTSOSJp9ioQ=\\\u0022,\\\u0022iv\\\u0022:\\\u0022EmjDVoGh70OoaI\\\\\\\/tXLsI2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022bc0099f1bdc1a577b86608c137b85f155b9b1500b17dcee20ec1e522e57cdf01\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2023.06.16.545386v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/biorxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"account":["G-RZD586MC3Q"],"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-3222388 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/biorxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2023.06.16.545386v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="search_rightsidebar_keywords_1118686836">Search for this keyword </label>
 <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1118686836" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><span class="icon-search"></span><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_172860612" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ETnPX892Rszo-7FMfIOX-YTMNXl1mKMKqIr6tSa4DBQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article node3222388" data-node-nid="3222388" id="node-3222388--2266106099" data-pisa="biorxiv;2023.06.16.545386v2" data-pisa-master="biorxiv;2023.06.16.545386" data-apath="/biorxiv/early/2023/06/26/2023.06.16.545386.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Avi J.</span> <span class="nlm-surname">Samelson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nabeela</span> <span class="nlm-surname">Ariqat</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">McKetney</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gita</span> <span class="nlm-surname">Rohanitazangi</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Celeste Parra</span> <span class="nlm-surname">Bravo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Darrin</span> <span class="nlm-surname">Goodness</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ruilin</span> <span class="nlm-surname">Tian</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Parker</span> <span class="nlm-surname">Grosjean</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Romany</span> <span class="nlm-surname">Abskharon</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">David</span> <span class="nlm-surname">Eisenberg</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Nicholas M.</span> <span class="nlm-surname">Kanaan</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Li</span> <span class="nlm-surname">Gan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Carlo</span> <span class="nlm-surname">Condello</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Danielle L.</span> <span class="nlm-surname">Swaney</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0002-3819-7019" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><span class="hw-icon-orcid hw-icon-color-orcid"></span> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Kampmann</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2023.06.16.545386 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node-3222388--2266106099" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Avi J. Samelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Avi%2BJ.%2BSamelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samelson%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAvi%2BJ.%2BSamelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nabeela Ariqat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nabeela%2BAriqat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ariqat%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANabeela%2BAriqat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin McKetney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>University of California San Francisco, Quantitative Biosciences Institute (QBI)</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>University of California San Francisco, Department of Cellular and Molecular Pharmacology</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Gladstone Institute of Data Science and Biotechnology, Gladstone Institutes</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BMcKetney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKetney%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BMcKetney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gita Rohanitazangi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gita%2BRohanitazangi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rohanitazangi%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGita%2BRohanitazangi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Celeste Parra Bravo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Helen and Robert Appel Alzheimer Disease Research Institute, Feil Family Brain and Mind Research Institute, Weill Cornell Medicine</span>, New York, NY, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celeste%2BParra%2BBravo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bravo%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACeleste%2BParra%2BBravo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Darrin Goodness </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darrin%2BGoodness%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodness%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarrin%2BGoodness%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ruilin Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ruilin%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARuilin%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Parker Grosjean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parker%2BGrosjean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grosjean%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParker%2BGrosjean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Romany Abskharon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Departments of Chemistry and Biochemistry and Biological Chemistry, UCLA-DOE Institute, UCLA</span>, Los Angeles, CA <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Romany%2BAbskharon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abskharon%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARomany%2BAbskharon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">David Eisenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Departments of Chemistry and Biochemistry and Biological Chemistry, UCLA-DOE Institute, UCLA</span>, Los Angeles, CA <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Howard Hughes Medical Institute UCLA</span>, Los Angeles, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BEisenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eisenberg%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BEisenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Nicholas M. Kanaan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Translational Neuroscience, Michigan State University</span>, East Lansing, MI, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BM.%2BKanaan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kanaan%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BM.%2BKanaan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Li Gan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Helen and Robert Appel Alzheimer Disease Research Institute, Feil Family Brain and Mind Research Institute, Weill Cornell Medicine</span>, New York, NY, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BGan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALi%2BGan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Carlo Condello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Department of Neurology, University of California San Francisco</span>, San Francisco, ca</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlo%2BCondello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Condello%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlo%2BCondello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Danielle L. Swaney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>University of California San Francisco, Quantitative Biosciences Institute (QBI)</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>University of California San Francisco, Department of Cellular and Molecular Pharmacology</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Gladstone Institute of Data Science and Biotechnology, Gladstone Institutes</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danielle%2BL.%2BSwaney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Swaney%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanielle%2BL.%2BSwaney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Martin Kampmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Neurodegenerative Diseases, University of California</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Department of Biochemistry and Biophysics, University of California San Francisco</span>, San Francisco, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BKampmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kampmann%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BKampmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3819-7019" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin Kampmann</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martin.kampmann@ucsf.edu" class="" data-icon-position="" data-hide-link-title="0">martin.kampmann@ucsf.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2023.06.16.545386v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:3222388" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:3222388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2023.06.16.545386v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:3222388" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:3222388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2023.06.16.545386v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:3222388" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:3222388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2023.06.16.545386v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:3222388" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:3222388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2023.06.16.545386v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:3222388" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:3222388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2023.06.16.545386v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:3222388" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:3222388/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><p id="p-3">A hallmark of age-associated neurodegenerative diseases is the aggregation of proteins. Aggregation of the protein tau defines tauopathies, which include Alzheimer’s disease and frontotemporal dementia. Specific neuronal subtypes are selectively vulnerable to the accumulation of tau aggregates, and subsequent dysfunction and death. The mechanisms underlying cell type-selective vulnerability are unknown. To systematically uncover the cellular factors controlling the accumulation of tau aggregates in human neurons, we conducted a genome-wide CRISPRi-based modifier screen in iPSC-derived neurons. The screen uncovered expected pathways, including autophagy, but also unexpected pathways including UFMylation and GPI anchor synthesis, that control tau oligomer levels. We identify the E3 ubiquitin ligase CUL5 as a tau interactor and potent modifier of tau levels. In addition, disruption of mitochondrial function increases tau oligomer levels and promotes proteasomal misprocessing of tau. These results reveal new principles of tau proteostasis in human neurons and pinpoint potential therapeutic targets for tauopathies.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">Tauopathies, which include Alzheimer’s disease, are widespread neurodegenerative diseases defined by pathological aggregation of the protein tau and there are limited therapeutics for these diseases. Tau is an intrinsically disordered protein that has six splice isoforms and is known to interact with microtubules. More recent work has established the role of tau in diverse neuronal processes, including axonal transport and synaptic transmission<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. Mutations in the gene that encodes tau, <em>MAPT</em>, cause familial forms of frontotemporal lobar degeneration (FTLD)<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>–<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. Most tauopathies, however, are not familial<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>, suggesting that factors in the cellular environment contribute to tauopathy onset.</p><p id="p-7">A key characteristic of tauopathies is selective vulnerability: specific regions of the brain and neuronal subtypes within them are vulnerable to specific tauopathies<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>, again pointing to the importance of the cellular environment for tau aggregation. Furthermore, recent structures of tau aggregates from patients have revealed disease-specific tau aggregate structures<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>, suggesting that determinants of tau conformation in different cellular environments may drive distinct pathological conformation and disease outcomes. Tau is also known to be highly post-translationally modified, and specific post-translational modifications are associated with disease<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>, providing a direct link between tau primary sequence and differential expression or activity of factors in the cellular environment.</p><p id="p-8">A major challenge has been to identify the specific cellular factors contributing to early changes in tau conformation that can lead to tau misfolding and cellular toxicity and dysfunction. GWAS studies<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> uncover modifiers of disease risk but do not provide molecular mechanisms. Similarly, single-cell transcriptomics<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>, can describe the factors differentially expressed in vulnerable versus resilient neuronal subtypes, but these studies lack direct experiments to pinpoint those factors that causally control pathology.</p><p id="p-9">Experimental model systems enable the mechanistic dissection of factors controlling tau aggregation, but some previously described models have limitations in terms of physiological relevance (such as tau over-expression in non-neuronal cell types) or are not amenable to high-throughput functional experiments that are required to gain a comprehensive understanding of factors determining tau conformational control in human neurons.</p><p id="p-10">Here, we use tau conformational-specific antibodies as probes for CRISPR-based genetic modifier screens in iPSC-derived human neurons<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> harboring the mutation <em>MAPT</em> V337M, which is 100% penetrant for FTLD<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. We utilize this new screening methodology to perform a genome-wide screen for tau oligomers. Tau oligomers are formed during the early stages of aggregation and there is some evidence that they are more toxic than larger protein aggregates<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. We then perform several secondary screens to compare modifiers of tau levels and tau oligomers in isogenic pairs of iPSC-derived neurons harboring the <em>MAPT</em> V337M and WT genotypes.</p><p id="p-11">We find that the strongest single gene shared among these screens, <em>CUL5</em>, is a new tau interactor that controls tau levels. We follow up on the single strongest class of genes–those genes involved in mitochondrial function– and find that acute oxidative stress induces a proteasome-derived tau fragment. This fragment is very similar to many tauopathy cerebrospinal fluid (CSF) biomarkers, suggesting that these biomarkers could be a signature of oxidative stress induced proteasome dysfunction. Together, these data provide a comprehensive and unbiased view of cellular factors that control tau levels and oligomerization in human neurons.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>MAPT V337M neurons accumulate tau oligomers</h3><p id="p-12">Since iPSC-derived models of tauopathy do not spontaneously accumulate mature pathological forms of tau, such as neurofibrillary tangles, we decided to characterize early stages of aggregation. Specifically, tau oligomers that can be detected by conformation-specific antibodies. We compared the binding of conformation-specific antibodies between isogenic <em>MAPT</em> WT (hereafter referred to as WT) and <em>MAPT</em> V337M (hereafter referred to as V337M) iPSC-derived neurons at 14 days post-differentiation. Tau oligomer levels were higher in V337M neurons than WT neurons, as measured by the oligomer-specific antibody T22<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup> using flow cytometry (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A-B</a>). Importantly, knockdown of <em>MAPT</em> using CRISPRi reduced levels of T22 staining, confirming the dependence of T22 levels on tau expression. Furthermore, we observed a dose-dependent effect of T22 staining dependent on the expression of one (<em>MAPT</em> WT/V337M) or two (<em>MAPT</em> V337M/V337M) copies of the V337M tau variant.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MAPT V337M neurons have higher levels of tau oligomers. (A) Left, Tau oligomer levels measured using the antibody T22 by flow cytometry in isogenic iPSC-derived neurons with one or two copies of the dementia-causing MAPT V337M mutation. MAPT knockdown neurons (grey) are included to establish background fluorescence signal. Intensities were normalized to the average of WT tau neurons (black). Right, Representative histograms are shown. (B) Left, Tau levels measured using the total tau antibody tau13 by flow cytometry in the same panel of neurons. Intensities were normalized to the average of WT tau neurons (black). Right, representative histograms are shown. Two-way ANOVA was used for all statistical analysis. All samples are the average of twelve biological replicates, error bars are &#xB1;standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;&lt;em&gt;MAPT V337M&lt;/em&gt; neurons have higher levels of tau oligomers.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; &lt;em&gt;Left,&lt;/em&gt; Tau oligomer levels measured using the antibody T22 by flow cytometry in isogenic iPSC-derived neurons with one or two copies of the dementia-causing &lt;em&gt;MAPT&lt;/em&gt; V337M mutation. &lt;em&gt;MAPT&lt;/em&gt; knockdown neurons (grey) are included to establish background fluorescence signal. Intensities were normalized to the average of WT tau neurons (black). &lt;em&gt;Right,&lt;/em&gt; Representative histograms are shown. &lt;strong&gt;(B)&lt;/strong&gt; &lt;em&gt;Left,&lt;/em&gt; Tau levels measured using the total tau antibody tau13 by flow cytometry in the same panel of neurons. Intensities were normalized to the average of WT tau neurons (black). &lt;em&gt;Right,&lt;/em&gt; representative histograms are shown. Two-way ANOVA was used for all statistical analysis. All samples are the average of twelve biological replicates, error bars are &#xB1;standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F1.medium.gif" width="440" height="161"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F1.medium.gif" width="440" height="161"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title"><em>MAPT V337M</em> neurons have higher levels of tau oligomers.</span><p id="p-13" class="first-child"><strong>(A)</strong> <em>Left,</em> Tau oligomer levels measured using the antibody T22 by flow cytometry in isogenic iPSC-derived neurons with one or two copies of the dementia-causing <em>MAPT</em> V337M mutation. <em>MAPT</em> knockdown neurons (grey) are included to establish background fluorescence signal. Intensities were normalized to the average of WT tau neurons (black). <em>Right,</em> Representative histograms are shown. <strong>(B)</strong> <em>Left,</em> Tau levels measured using the total tau antibody tau13 by flow cytometry in the same panel of neurons. Intensities were normalized to the average of WT tau neurons (black). <em>Right,</em> representative histograms are shown. Two-way ANOVA was used for all statistical analysis. All samples are the average of twelve biological replicates, error bars are ±standard deviation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Genome-wide screen for modifiers of tau oligomer levels</h3><p id="p-14">To enable a genome-wide modifier for screen for tau oligomer levels in V337M neurons, we engineered the V337M iPSC line to express CRISPRi machinery and optimized a protocol (see methods) for the FACS sorting of fixed iPSC-derived neurons using the anti-tau oligomer antibody T22 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Briefly, we infected iPSCs with a lentiviral CRISPRi sgRNA library targeting all protein coding genes (five sgRNAs per gene, 104,535 sgRNAs) and 1,895 non-targeting controls<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>, differentiated them into neurons, and fixed them at Day 14 post-differentiation. Neurons were then stained with T22 antibody and an antibody against the neuronal marker NeuN. NeuN-positive cells were FACS sorted into two bins: those neurons that had the highest and the lowest thirty percent of T22 signal. Frequencies of sgRNAs in each bin were determined by next-generation sequencing and compared to NTCs using a Mann-Whitney U test to assign p-values for each gene (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). 1,143 genes were called hits based on a false discovery rate (FDR) of 5% (see Methods).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide screen for tau oligomers levels in iPSC-derived neurons. (A) Schematic of screen. Tau V337M heterozygous iPSCs were transduced with a pooled library of sgRNAs targeting every protein-coding gene. iPSCs were differentiated into excitatory neurons for two-weeks, fixed, and stained with T22 and NeuN. The thirty percent of NeuN-positive cells with the lowest (blue) and highest (red) tau oligomer signal were separated by FACS sorting. Genomic DNA was isolated from each population and the sgRNA cassette was sequenced using next-generation sequencing. Comparison of sgRNA frequencies was used to call hit genes. (B) Volcano plot of hit genes from genome-wide screen. Phenotype (normalized log2 of T22 High versus T22 low counts), is plotted versus the negative log of the P-value, calculated with a Mann-Whitney U-test. Positive hits are in pink, and negative in light blue. Non-targeting controls are in black and non-hit genes are in grey. MAPT is a top hit (yellow). Legend continues over-leaf. Groups of genes as identified by KEGG Pathway analysis (C) are highlighted: GPI-anchor (blue), Ubiquitin Proteasome System (Purple), Autophagy and mTOR (green), Mitochondrial genes (red), and UFMylation genes (orange). (C) KEGG Pathway analysis was performed after removal of mitochondrial genes. Top ten pathways by adjusted p-value are listed for hits knockdown of which increase tau oligomer levels (light pink) and for hits knockdown of which decrease tau oligomer levels (light blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genome-wide screen for tau oligomers levels in iPSC-derived neurons.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Schematic of screen. Tau V337M heterozygous iPSCs were transduced with a pooled library of sgRNAs targeting every protein-coding gene. iPSCs were differentiated into excitatory neurons for two-weeks, fixed, and stained with T22 and NeuN. The thirty percent of NeuN-positive cells with the lowest (blue) and highest (red) tau oligomer signal were separated by FACS sorting. Genomic DNA was isolated from each population and the sgRNA cassette was sequenced using next-generation sequencing. Comparison of sgRNA frequencies was used to call hit genes. &lt;strong&gt;(B)&lt;/strong&gt; Volcano plot of hit genes from genome-wide screen. Phenotype (normalized log2 of T22 High versus T22 low counts), is plotted versus the negative log of the P-value, calculated with a Mann-Whitney U-test. Positive hits are in pink, and negative in light blue. Non-targeting controls are in black and non-hit genes are in grey. &lt;em&gt;MAPT&lt;/em&gt; is a top hit (yellow). &lt;strong&gt;Legend continues over-leaf.&lt;/strong&gt; Groups of genes as identified by KEGG Pathway analysis &lt;strong&gt;(C)&lt;/strong&gt; are highlighted: GPI-anchor (blue), Ubiquitin Proteasome System (Purple), Autophagy and mTOR (green), Mitochondrial genes (red), and UFMylation genes (orange). &lt;strong&gt;(C)&lt;/strong&gt; KEGG Pathway analysis was performed after removal of mitochondrial genes. Top ten pathways by adjusted p-value are listed for hits knockdown of which increase tau oligomer levels (light pink) and for hits knockdown of which decrease tau oligomer levels (light blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F2.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F2.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Genome-wide screen for tau oligomers levels in iPSC-derived neurons.</span><p id="p-15" class="first-child"><strong>(A)</strong> Schematic of screen. Tau V337M heterozygous iPSCs were transduced with a pooled library of sgRNAs targeting every protein-coding gene. iPSCs were differentiated into excitatory neurons for two-weeks, fixed, and stained with T22 and NeuN. The thirty percent of NeuN-positive cells with the lowest (blue) and highest (red) tau oligomer signal were separated by FACS sorting. Genomic DNA was isolated from each population and the sgRNA cassette was sequenced using next-generation sequencing. Comparison of sgRNA frequencies was used to call hit genes. <strong>(B)</strong> Volcano plot of hit genes from genome-wide screen. Phenotype (normalized log2 of T22 High versus T22 low counts), is plotted versus the negative log of the P-value, calculated with a Mann-Whitney U-test. Positive hits are in pink, and negative in light blue. Non-targeting controls are in black and non-hit genes are in grey. <em>MAPT</em> is a top hit (yellow). <strong>Legend continues over-leaf.</strong> Groups of genes as identified by KEGG Pathway analysis <strong>(C)</strong> are highlighted: GPI-anchor (blue), Ubiquitin Proteasome System (Purple), Autophagy and mTOR (green), Mitochondrial genes (red), and UFMylation genes (orange). <strong>(C)</strong> KEGG Pathway analysis was performed after removal of mitochondrial genes. Top ten pathways by adjusted p-value are listed for hits knockdown of which increase tau oligomer levels (light pink) and for hits knockdown of which decrease tau oligomer levels (light blue).</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Knockdown of <em>MAPT</em> was a top hit that decreased tau oligomer levels, as expected. Pathway analysis of hit genes revealed several pathways expected based on the previous literature. Known regulators of tau and tau oligomer levels, such as genes that control autophagy, were significantly enriched (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B-D</a>), with knockdown of the autophagic machinery and positive regulators of autophagy, including <em>WIPI2, ATG14, PIK3C3, ATG101,</em> and <em>LAMTOR1,3</em> and <em>5</em> increasing tau oligomer levels, while knockdown of negative regulators of autophagy, such as <em>MTOR</em>, decreased tau oligomer levels<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>–<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Recent studies have implicated methyl-6-Adenosine (m<sup>6</sup>A) as a modulator of tau oligomerization<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> and knockdown of genes involved in the regulation and deposition of m<sup>6</sup>A, including <em>METTL14, METTL3,</em> and <em>HNRNPA1</em>, were among the tops hits that decrease tau levels upon knockdown. These results confirm that the methodology used here recapitulates shared characteristics of tau biology in cell culture, mouse models, and human samples.</p><p id="p-17">The strongest gene signature overall in this screen are genes involved in mitochondrial function, especially the electron transport chain (ETC). Knockdown of these genes strongly increased tau oligomer levels (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2B</a>). KEGG pathway analysis without mitochondrial hits (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>) revealed other pathways controlling tau oligomer levels. As mentioned above, knockdown of positive regulators of autophagy and lysosomal degradation were significantly enriched as top hits that increase tau oligomer levels. Factors from the other branch of protein degradation, the Ubiquitin/Proteasome System (UPS) were also among the top hits in this screen, including <em>VHL</em>, the proteasome chaperones <em>PSMG1</em> and <em>PSMG3</em>, the proteasomal de-ubiquitinase <em>PSMD14</em> (also known as rpn11), and several E3, E2, and E1 ubiquitin ligases. The Cullin-RING ligase (CRL) <em>CUL5</em>, and its obligate E2 ligase adaptor, <em>RNF7</em>, were also among top hits knockdown of which increases tau oligomer levels.</p><p id="p-18">KEGG pathway analysis of genes knockdown of which decreased tau oligomer levels included negative regulators of autophagy, including the genes that encode the mTORC1 subunits mTOR, RAPTOR, and MLST8, which inhibit autophagy through ULK1. Surprisingly, GPI-anchor biosynthesis was the most significantly enriched set of genes whose knockdown decreases tau oligomer levels. Knockdown of genes including <em>ACAT2</em>, <em>PMVK</em>, <em>IDI1</em>, and <em>FDPS</em>, which are involved in synthesis of mevalonate, a precursor to molecules such as cholesterol, also decreased tau oligomer levels. Finally, knockdown of genes essential for UFMylation, including <em>UFM1</em>, <em>UFL1</em>, and <em>DDRGK1</em>, decreased tau oligomer levels. In a related manuscript, we validate UFMylation as a pathway controlling levels of tau aggregates<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>.</p></div><div id="sec-5" class="subsection"><h3>Secondary screens pinpoint cellular factors controlling tau levels and genotype-specific factors</h3><p id="p-19">To characterize the top hits more closely, we cloned a small, pooled library targeting 1,037 hit genes (5 sgRNAs per gene) and 250 non-targeting controls and performed seven CRISPRi secondary screens using this library (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><p id="p-20">The first three screens used two alternative tau oligomer antibodies, TOC1<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> and M204<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>, as well as retesting T22. There was a substantial overlap in modifiers of tau oligomers detected by all three antibodies. KEGG Pathway analysis of the 106 shared hits knockdown of which increased oligomer levels detected by all three antibodies confirmed that autophagy, mTOR signalling, and the UPS system are high-confidence gene sets that control tau oligomer levels (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B-C</a>). Conversely, GPI-anchor biosynthesis and RNA transport were pathways enriched among genes knockdown of which decreases tau oligomer levels detected by all three antibodies (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Secondary screens reveal modifiers specific to tau oligomer levels and MAPT genotype (A) Schematic of retest strategy. A focused sgRNA library targeting all the hits from the genome-wide screen was screened in MAPT WT and V337M neurons using a panel of different antibodies. (B) Overlapping hit genes knockdown of which increases tau oligomer levels between three tau oligomer antibodies, T22, TOC1, and M204. Top KEGG Pathways of overlapping genes are listed after removal of mitochondrial genes. (C) Overlapping hit genes knockdown of which decreases tau oligomer levels between three tau oligomer antibodies, T22, TOC1, and M204. Top KEGG Pathways of the overlapping genes are listed. Legend continues over-leaf. (D) Comparison of total tau and tau oligomer screens in V337M neurons by gene score. ETC Complex I genes (red), ER/UFMylation genes (orange), and GPI-anchor genes (blue) were oligomer specific hits. CUL5 is in purple. mTOR and AMPK signaling genes are in green. (E) Comparison of WT and V337M tau oligomer screens by gene score. GPI-anchor genes (blue) were V337M specific, while knockdown of genes involved in mRNA transport (white), especially nuclear pore subunits, strongly decreased tau oligomer levels in WT but not V337M neurons. Knockdown TCA cycle genes (red) and genes involved in AMPK signaling (green) weakly increased tau oligomers in WT, but not V337M neurons. CUL5 is in purple. (F) Comparison of screens for total tau levels in MAPT V337M versus WT neurons. V337M total tau levels were more sensitive to knockdown of mTOR signaling (green) and GPI anchor genes (blue). WT total tau levels were more sensitive to knockdown of genes involved in RNA degradation (white). CUL5 is in purple." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Secondary screens reveal modifiers specific to tau oligomer levels and MAPT genotype&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Schematic of retest strategy. A focused sgRNA library targeting all the hits from the genome-wide screen was screened in &lt;em&gt;MAPT&lt;/em&gt; WT and V337M neurons using a panel of different antibodies. &lt;strong&gt;(B)&lt;/strong&gt; Overlapping hit genes knockdown of which increases tau oligomer levels between three tau oligomer antibodies, T22, TOC1, and M204. Top KEGG Pathways of overlapping genes are listed after removal of mitochondrial genes. &lt;strong&gt;(C)&lt;/strong&gt; Overlapping hit genes knockdown of which decreases tau oligomer levels between three tau oligomer antibodies, T22, TOC1, and M204. Top KEGG Pathways of the overlapping genes are listed. &lt;strong&gt;Legend continues over-leaf&lt;/strong&gt;. &lt;strong&gt;(D)&lt;/strong&gt; Comparison of total tau and tau oligomer screens in V337M neurons by gene score. ETC Complex I genes (red), ER/UFMylation genes (orange), and GPI-anchor genes (blue) were oligomer specific hits. &lt;em&gt;CUL5&lt;/em&gt; is in purple. mTOR and AMPK signaling genes are in green. &lt;strong&gt;(E)&lt;/strong&gt; Comparison of WT and V337M tau oligomer screens by gene score. GPI-anchor genes (blue) were V337M specific, while knockdown of genes involved in mRNA transport (white), especially nuclear pore subunits, strongly decreased tau oligomer levels in WT but not V337M neurons. Knockdown TCA cycle genes (red) and genes involved in AMPK signaling (green) weakly increased tau oligomers in WT, but not V337M neurons. &lt;em&gt;CUL5&lt;/em&gt; is in purple. &lt;strong&gt;(F)&lt;/strong&gt; Comparison of screens for total tau levels in MAPT V337M versus WT neurons. V337M total tau levels were more sensitive to knockdown of mTOR signaling (green) and GPI anchor genes (blue). WT total tau levels were more sensitive to knockdown of genes involved in RNA degradation (white). &lt;em&gt;CUL5&lt;/em&gt; is in purple.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F3.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F3.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">Secondary screens reveal modifiers specific to tau oligomer levels and MAPT genotype</span><p id="p-21" class="first-child"><strong>(A)</strong> Schematic of retest strategy. A focused sgRNA library targeting all the hits from the genome-wide screen was screened in <em>MAPT</em> WT and V337M neurons using a panel of different antibodies. <strong>(B)</strong> Overlapping hit genes knockdown of which increases tau oligomer levels between three tau oligomer antibodies, T22, TOC1, and M204. Top KEGG Pathways of overlapping genes are listed after removal of mitochondrial genes. <strong>(C)</strong> Overlapping hit genes knockdown of which decreases tau oligomer levels between three tau oligomer antibodies, T22, TOC1, and M204. Top KEGG Pathways of the overlapping genes are listed. <strong>Legend continues over-leaf</strong>. <strong>(D)</strong> Comparison of total tau and tau oligomer screens in V337M neurons by gene score. ETC Complex I genes (red), ER/UFMylation genes (orange), and GPI-anchor genes (blue) were oligomer specific hits. <em>CUL5</em> is in purple. mTOR and AMPK signaling genes are in green. <strong>(E)</strong> Comparison of WT and V337M tau oligomer screens by gene score. GPI-anchor genes (blue) were V337M specific, while knockdown of genes involved in mRNA transport (white), especially nuclear pore subunits, strongly decreased tau oligomer levels in WT but not V337M neurons. Knockdown TCA cycle genes (red) and genes involved in AMPK signaling (green) weakly increased tau oligomers in WT, but not V337M neurons. <em>CUL5</em> is in purple. <strong>(F)</strong> Comparison of screens for total tau levels in MAPT V337M versus WT neurons. V337M total tau levels were more sensitive to knockdown of mTOR signaling (green) and GPI anchor genes (blue). WT total tau levels were more sensitive to knockdown of genes involved in RNA degradation (white). <em>CUL5</em> is in purple.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We next screened both the WT and V337M lines using a total tau antibody, and the WT line using T22, to find hits that uniquely control tau overall or tau oligomer levels, as well as hits unique to V337M, respectively (Figure D-F). ETC complex I hits where unique to the tau oligomer screens, as were GPI-anchor biosynthesis hits and ER/UFMylation genes. The WT tau line seemed to be more sensitive to knockdown of genes involved in mRNA transport, while the V337M line was enriched for hits that regulate mTOR. Finally, the E3 ubiquitin ligase CUL5 was a top hit in all screens we performed regardless of tau mutation or antibody. Intriguingly, CUL5 is also a gene expressed more highly in excitatory neurons in the human entorhinal cortex that were resilient to Alzheimer’s disease, compared to vulnerable neurons in a scRNA-seq study performed by our lab<sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>), suggesting that it may causally contribute to selective vulnerability in the context of Alzheimer’s disease.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CUL5 regulates tau levels. (A) Comparison between genes expressed more highly in excitatory neurons in the human entorhinal cortex that are resilient versus vulnerable to Alzheimer&#x2019;s diseases 11 and genes that increase tau oligomer levels in this study. (B) Schematic of CUL5 E3 Ligase complexes. (C,D) Individual knockdown of CUL5 and RNF7 reveals increases in Tau oligomer and total tau levels by flow cytometry (B) and western blot (C). Average of twelve biological replicates (E) CUL5-flag co-elutes with tau, as well as other members of CUL5 complexes, elongin B (ELOB) and RNF7. (F) Ratiometric reporter assay for determining CUL5-dependent degrons in tau (top). Constructs tiling the tau sequence are shown in different colors, with amino acid numbers referring to the 0N3R sequence (middle). Results from the ratiometric reporter in cells expressing non-targeting (NTC) sgRNAs or sgRNAs targeting Cul5. Only tau itself and tile 3 (residues 80-130) increase in levels upon CUL5 knockdown and this effect is abrogated upon treatment with the proteasome inhibitor carfilzomib at 10nM (G). (H) Volcano plot as in Figure 2, but with all known CUL5 adaptors colored in purple, as well as the CUL5, RNF7, and the CUL5 interactor ARIH2. (I) Tau oligomer (left) and total tau levels (right) measured during overexpression of SOCS4 using CRISPRa. Average of seven biological replicates. Two-way ANOVA was used for all statistical analysis. All data points are the average of six biological replicates, unless otherwise stated. Error bars are &#xB1;standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;CUL5 regulates tau levels.&lt;/span&gt; &lt;strong&gt;(A) C&lt;/strong&gt;omparison between genes expressed more highly in excitatory neurons in the human entorhinal cortex that are resilient versus vulnerable to Alzheimer&#x2019;s diseases &lt;sup&gt;11&lt;/sup&gt; and genes that increase tau oligomer levels in this study. &lt;strong&gt;(B)&lt;/strong&gt; Schematic of CUL5 E3 Ligase complexes. &lt;strong&gt;(C,D)&lt;/strong&gt; Individual knockdown of CUL5 and RNF7 reveals increases in Tau oligomer and total tau levels by flow cytometry (B) and western blot (C). Average of twelve biological replicates &lt;strong&gt;(E)&lt;/strong&gt; CUL5-flag co-elutes with tau, as well as other members of CUL5 complexes, elongin B (ELOB) and RNF7. &lt;strong&gt;(F)&lt;/strong&gt; Ratiometric reporter assay for determining CUL5-dependent degrons in tau (top). Constructs tiling the tau sequence are shown in different colors, with amino acid numbers referring to the 0N3R sequence (middle). Results from the ratiometric reporter in cells expressing non-targeting (NTC) sgRNAs or sgRNAs targeting Cul5. Only tau itself and tile 3 (residues 80-130) increase in levels upon &lt;em&gt;CUL5&lt;/em&gt; knockdown and this effect is abrogated upon treatment with the proteasome inhibitor carfilzomib at 10nM &lt;strong&gt;(G)&lt;/strong&gt;. &lt;strong&gt;(H)&lt;/strong&gt; Volcano plot as in Figure 2, but with all known &lt;em&gt;CUL5&lt;/em&gt; adaptors colored in purple, as well as the &lt;em&gt;CUL5&lt;/em&gt;, &lt;em&gt;RNF7&lt;/em&gt;, and the &lt;em&gt;CUL5&lt;/em&gt; interactor &lt;em&gt;ARIH2&lt;/em&gt;. &lt;strong&gt;(I)&lt;/strong&gt; Tau oligomer (left) and total tau levels (right) measured during overexpression of SOCS4 using CRISPRa. Average of seven biological replicates. Two-way ANOVA was used for all statistical analysis. All data points are the average of six biological replicates, unless otherwise stated. Error bars are &#xB1;standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F4.medium.gif" width="341" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F4.medium.gif" width="341" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">CUL5 regulates tau levels.</span><p id="p-23" class="first-child"><strong>(A) C</strong>omparison between genes expressed more highly in excitatory neurons in the human entorhinal cortex that are resilient versus vulnerable to Alzheimer’s diseases <sup><a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a></sup> and genes that increase tau oligomer levels in this study. <strong>(B)</strong> Schematic of CUL5 E3 Ligase complexes. <strong>(C,D)</strong> Individual knockdown of CUL5 and RNF7 reveals increases in Tau oligomer and total tau levels by flow cytometry (B) and western blot (C). Average of twelve biological replicates <strong>(E)</strong> CUL5-flag co-elutes with tau, as well as other members of CUL5 complexes, elongin B (ELOB) and RNF7. <strong>(F)</strong> Ratiometric reporter assay for determining CUL5-dependent degrons in tau (top). Constructs tiling the tau sequence are shown in different colors, with amino acid numbers referring to the 0N3R sequence (middle). Results from the ratiometric reporter in cells expressing non-targeting (NTC) sgRNAs or sgRNAs targeting Cul5. Only tau itself and tile 3 (residues 80-130) increase in levels upon <em>CUL5</em> knockdown and this effect is abrogated upon treatment with the proteasome inhibitor carfilzomib at 10nM <strong>(G)</strong>. <strong>(H)</strong> Volcano plot as in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, but with all known <em>CUL5</em> adaptors colored in purple, as well as the <em>CUL5</em>, <em>RNF7</em>, and the <em>CUL5</em> interactor <em>ARIH2</em>. <strong>(I)</strong> Tau oligomer (left) and total tau levels (right) measured during overexpression of SOCS4 using CRISPRa. Average of seven biological replicates. Two-way ANOVA was used for all statistical analysis. All data points are the average of six biological replicates, unless otherwise stated. Error bars are ±standard deviation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>UL5 controls tau levels via tau ubiquitination</h3><p id="p-24">CUL5 is a Cullin-RING (CRL) E3 ubiquitin ligase that is best known for its role in regulating aspects of viral infection, especially for HIV<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>. CUL5 serves as a scaffold for facilitating ubiquitin transfer from an E2 ligase to CUL5 substrates. RNF7, also a strong hit in our screen (see <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>), is essential for binding of E2s to CUL5. The N-terminal region of CUL5 binds to substrates via modular substrate adaptors that are bound to CUL5 with the proteins Elongin B and Elongin C (ELOB and ELOC, also known as TCEB2 and TCEB3) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Activity of CUL5 is dependent on neddylation, a ubiquitin-like protein that is conjugated by the NEDD8-specific E2 UBE2F.</p><p id="p-25">Knockdown of both <em>CUL5</em> and <em>RNF7</em> with independently cloned sgRNAs recapitulated screen phenotypes by flow cytometry and western blot (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C-D</a>). Western blot showed smaller increases in tau levels, possibly because flow cytometry only measures tau levels in the neuron soma and not in axons. We then over-expressed CUL5-3xFLAG to ask if CUL5 and tau physically interact. Immunoprecipitation of CUL5-3xFLAG, but not GFP-3xFLAG, and subsequent FLAG elution revealed tau in the eluate (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4E</a>). Importantly, ELOB and RNF7 were both present in the eluate, confirming that functional CUL5 complexes were immunoprecipitated.</p><p id="p-26">Cullin-ring ligases (CRLs) are known to recognize substrates via degrons, short amino acid sequences that bind to CRL substrate adaptors. To find the region of tau that acts as a CUL5 degron, we created N-terminal GFP fusions of eight tau “tiles” using a sliding window of fifty amino acids with a ten amino acid overlap and fused them to a T2A-mApple sequence to create a ratiometric reporter of tau tile levels (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4F</a>). We made stable lines expressing these tiles in iPSCs using lentivirus, transduced them with sgRNAs against <em>CUL5</em> or non-targeting controls, differentiated them into neurons and, 14 days post-differentiation, assayed the GFP/mApple ratio. Along with full length tau, only Tile 3, 0N3R tau residues 80-130 (2N4R tau residues 138-188) is sensitive to <em>CUL5</em> knockdown. This increase in Tile 3 levels upon <em>CUL5</em> knockdown was dependent on proteasome activity, as treatment with the proteasome inhibitor Carfilzomib abrogated this effect (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4G</a>).</p><p id="p-27">CRLs bind substrates via adaptor proteins that bind to the N-terminal region of the cullin. In our screen, two CUL5 adaptors were positive hits-SOCS4 and SOCS5 (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4H</a>). Over-expression of SOCS4 using CRISPRa decreased tau levels, suggesting that CUL5 binds tau via SOCS4 (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4I</a>).</p></div><div id="sec-7" class="subsection"><h3>Oxidative stress promotes the formation of a proteasome-derived N-terminal tau fragment</h3><p id="p-28">Genes essential for the function of mitochondria, as well as other genes such as <em>FECH, PSAP</em> and <em>FH,</em> knockdown of which we previously found to increase levels of reactive oxygen species (ROS) in neurons<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup> (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S1</a>) were all hits knockdown of which increased tau and tau oligomer levels in our primary and secondary screens. We validated these hits using a pharmacological approach to investigate the role mitochondrial function, especially the ETC, on tau oligomer levels.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between T22 and CellRox genome-wide CRISPRi screens. Gene set enrichment was performed for shared positive and negative hits using ENRICHR and genes in identified enriched pathways are labelled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Comparison between T22 and CellRox genome-wide CRISPRi screens.&lt;/span&gt; Gene set enrichment was performed for shared positive and negative hits using ENRICHR and genes in identified enriched pathways are labelled.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F7.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F7.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1</span> <span class="caption-title">Comparison between T22 and CellRox genome-wide CRISPRi screens.</span><p id="p-72" class="first-child">Gene set enrichment was performed for shared positive and negative hits using ENRICHR and genes in identified enriched pathways are labelled.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">We treated neurons with increasing concentrations of rotenone, an ETC complex I inhibitor, for 24 hours and measured tau and T22 levels by flow cytometry (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A-B</a>). This revealed a rotenone concentration-dependent increase in both tau and T22 levels in the V337M line. Therefore, all further experiments were done in the V337M line. Validation of these phenotypes by Western blot revealed that rotenone promoted the formation of a ∼25kD fragment of tau detected by the tau13 antibody, which recognizes tau residues 2-18 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5C</a>). Expression of a GFP-0N3R tau transgene and treatment with 200nM rotenone revealed the same molecular weight shift (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5D</a>), confirming that this fragment is due to post-transcriptional processing of tau.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Oxidative stress increases tau oligomer and total tau levels and induces a tau cleavage fragment. (A,B) Levels of tau oligomers (A) and total tau (B) increase in a rotenone concentration-dependent manner in V337M but only tau levels, and not tau oligomer levels increase in WT tau neurons. (C) Western blot of rotenone-treated neurons reveals a tau cleavage fragment. (D) Rotenone treatment of neurons results in an equivalent fragment from a GFP-Tau transgene. Fragment levels as measured by western blot (E) and ROS levels as measured by flow cytometry using CellROX (F) show similar rotenone concentration-dependent increases. (G) Treatment with 100&#xB5;M hydrogen peroxide (right) drastically increases fragment formation, while treatment with the antioxidant N-Acetyl-Cysteine at 1&#xB5;M (right) decreases fragment formation (quantitation far right). If not otherwise shown, rotenone concentration is at 200nM. Two-way ANOVA was used for all statistical analysis. All samples are the average of six biological replicates, error bars are &#xB1;standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Oxidative stress increases tau oligomer and total tau levels and induces a tau cleavage fragment.&lt;/span&gt; &lt;strong&gt;(A,B)&lt;/strong&gt; Levels of tau oligomers (A) and total tau (B) increase in a rotenone concentration-dependent manner in V337M but only tau levels, and not tau oligomer levels increase in WT tau neurons. &lt;strong&gt;(C)&lt;/strong&gt; Western blot of rotenone-treated neurons reveals a tau cleavage fragment. &lt;strong&gt;(D)&lt;/strong&gt; Rotenone treatment of neurons results in an equivalent fragment from a GFP-Tau transgene. Fragment levels as measured by western blot &lt;strong&gt;(E)&lt;/strong&gt; and ROS levels as measured by flow cytometry using CellROX &lt;strong&gt;(F)&lt;/strong&gt; show similar rotenone concentration-dependent increases. &lt;strong&gt;(G)&lt;/strong&gt; Treatment with 100&#xB5;M hydrogen peroxide (right) drastically increases fragment formation, while treatment with the antioxidant N-Acetyl-Cysteine at 1&#xB5;M (right) decreases fragment formation (quantitation far right). If not otherwise shown, rotenone concentration is at 200nM. Two-way ANOVA was used for all statistical analysis. All samples are the average of six biological replicates, error bars are &#xB1;standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F5.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F5.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <span class="caption-title">Oxidative stress increases tau oligomer and total tau levels and induces a tau cleavage fragment.</span><p id="p-30" class="first-child"><strong>(A,B)</strong> Levels of tau oligomers (A) and total tau (B) increase in a rotenone concentration-dependent manner in V337M but only tau levels, and not tau oligomer levels increase in WT tau neurons. <strong>(C)</strong> Western blot of rotenone-treated neurons reveals a tau cleavage fragment. <strong>(D)</strong> Rotenone treatment of neurons results in an equivalent fragment from a GFP-Tau transgene. Fragment levels as measured by western blot <strong>(E)</strong> and ROS levels as measured by flow cytometry using CellROX <strong>(F)</strong> show similar rotenone concentration-dependent increases. <strong>(G)</strong> Treatment with 100µM hydrogen peroxide (right) drastically increases fragment formation, while treatment with the antioxidant N-Acetyl-Cysteine at 1µM (right) decreases fragment formation (quantitation far right). If not otherwise shown, rotenone concentration is at 200nM. Two-way ANOVA was used for all statistical analysis. All samples are the average of six biological replicates, error bars are ±standard deviation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Measurement of ROS levels using CellRox, and 25 kD-fragment levels, by western blot, as a function of rotenone concentration revealed a concentration-dependent increase similar to that of T22 signal and tau levels (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5E-F</a>). Treatment with antimycin A, an ETC complex III inhibitor, but not CCCP, a proton-gradient uncoupler, or Oligomycin, an ATP Synthase inhibitor, also led to 25 kD-fragment formation (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S2</a>), suggesting that ROS production as a side effect of mitochondrial dysfunction induces 25 kD-fragment formation. Neurons treated with hydrogen peroxide revealed close to complete conversion of full-length tau into the 25 kD-fragment (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5G</a>). Concomitant treatment of neurons with rotenone and N-acetyl-cysteine, an antioxidant, decreased 25 kD-fragment levels as compared to rotenone alone (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5G</a>). Thus, acute increases in ROS levels generate this tau proteolytic fragment. 25 kD-fragment formation was not due to cell death via apoptosis or ferroptosis, as treatment with the pan-caspase inhibitor ZVAD-MVK, the ferroptosis inhibitor ferrostatin, or the ferroptosis inducer RSL3 did not change fragment levels (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure S3</a>). This experiment also rules out caspase cleavage of tau to generate the 25kD-fragment).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inhibition of ETC complex III, but not complex V or uncoupling of the mitochondrial protein gradient promotes fragment formation. Western blot of vehicle treated, antimycin A treated (complex III inhibitor), oligomycin treated (complex V inhibitor) or CCCP treated (protein gradient uncoupler). Data are three biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Inhibition of ETC complex III, but not complex V or uncoupling of the mitochondrial protein gradient promotes fragment formation.&lt;/span&gt; Western blot of vehicle treated, antimycin A treated (complex III inhibitor), oligomycin treated (complex V inhibitor) or CCCP treated (protein gradient uncoupler). Data are three biological replicates.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F8.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F8.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2</span> <span class="caption-title">Inhibition of ETC complex III, but not complex V or uncoupling of the mitochondrial protein gradient promotes fragment formation.</span><p id="p-73" class="first-child">Western blot of vehicle treated, antimycin A treated (complex III inhibitor), oligomycin treated (complex V inhibitor) or CCCP treated (protein gradient uncoupler). Data are three biological replicates.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Apoptosis and ferroptosis do not control fragment formation. (A) Western blot of rotenone-treated neurons with the ferroptosis inhibitor ferrostatin, or ferroptosis promoting molecule RSL3 reveals no changes to tau cleavage fragments. (C) Quantitation of data in (A). Western blot of rotenone-treated neurons with the pan-caspase inhibitor ZVAD-MVAK reveals no changes to tau cleavage fragments. (D) Quantitation of data in (C). Data are average of three biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Apoptosis and ferroptosis do not control fragment formation.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Western blot of rotenone-treated neurons with the ferroptosis inhibitor ferrostatin, or ferroptosis promoting molecule RSL3 reveals no changes to tau cleavage fragments. &lt;strong&gt;(C)&lt;/strong&gt; Quantitation of data in (A). Western blot of rotenone-treated neurons with the pan-caspase inhibitor ZVAD-MVAK reveals no changes to tau cleavage fragments. &lt;strong&gt;(D)&lt;/strong&gt; Quantitation of data in (C). Data are average of three biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F9.medium.gif" width="384" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F9.medium.gif" width="384" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3</span> <span class="caption-title">Apoptosis and ferroptosis do not control fragment formation.</span><p id="p-74" class="first-child"><strong>(A)</strong> Western blot of rotenone-treated neurons with the ferroptosis inhibitor ferrostatin, or ferroptosis promoting molecule RSL3 reveals no changes to tau cleavage fragments. <strong>(C)</strong> Quantitation of data in (A). Western blot of rotenone-treated neurons with the pan-caspase inhibitor ZVAD-MVAK reveals no changes to tau cleavage fragments. <strong>(D)</strong> Quantitation of data in (C). Data are average of three biological replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We next treated neurons with 200nM rotenone and different protease inhibitors to identify if any cellular proteases induced this cleavage fragment. We chose inhibitors targeting common tau proteases: cathepsins, calpain, and the proteasome<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>–<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup> (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S4</a>). Only inhibition of the proteasome decreased 25 kD-fragment formation (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>), suggesting that proteasomal processing or misprocessing creates this N-terminal tau fragment.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proteasome inhibition decreases fragment formation. (A) Western blot of neurons with the cathepsin inhibitor E64, calpain inhibitor calpastatin, and proteasome inhibitor MG132. (B) Western blot of rotenone treated neurons with the cathepsin inhibitor E64, calpain inhibitor calpastatin, and proteasome inhibitor MG132." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Proteasome inhibition decreases fragment formation.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Western blot of neurons with the cathepsin inhibitor E64, calpain inhibitor calpastatin, and proteasome inhibitor MG132. &lt;strong&gt;(B)&lt;/strong&gt; Western blot of rotenone treated neurons with the cathepsin inhibitor E64, calpain inhibitor calpastatin, and proteasome inhibitor MG132.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F10.medium.gif" width="395" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F10.medium.gif" width="395" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4</span> <span class="caption-title">Proteasome inhibition decreases fragment formation.</span><p id="p-75" class="first-child"><strong>(A)</strong> Western blot of neurons with the cathepsin inhibitor E64, calpain inhibitor calpastatin, and proteasome inhibitor MG132. <strong>(B)</strong> Western blot of rotenone treated neurons with the cathepsin inhibitor E64, calpain inhibitor calpastatin, and proteasome inhibitor MG132.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proteasome dysfunction causes accumulation of the tau cleavage fragment. (A) Treatment with proteasome inhibitor MG132 at 1&#xB5;M decreases levels of the 25kD rotenone-induced tau fragment. (B) Treatment with rotenone decreases proteasome activity (top) but not levels (bottom). (C) Quantitation of (B), average of six biological replicates. (D) Knockdown of PSME2 increases fragment levels (left) Western blot with quantitation. Over-expression with CRISPRa of PSME1 and PSME2 decrease fragment levels (right). Full gels in Figure S6. (E) Expression of GFP-tauMetless decreases fragment formation as compared to GFP-tau in neurons. Western blot (left), quantitation of western blot (right). (F) Traces of averaged intensity from peptides per amino acid derived from mass spectrometry data for full-length tau fragment (black) and rotenone induced fragment (dark red). A sharp decrease in intensity is seen in the rotenone induced fragment gel slice after residue 200, except for one peptide (see Figure S7). Stars represent neo-tryptic termini identified upon digestion with GluC, pinpointing two possible C-termini for the fragment, narrowing the fragment identity to residues 172-200 (2N4R numbering: 230-258). (G) Western blot of cell lysate (top) and undiluted neuron-conditioned media (bottom) shows fragment in the media. Rotenone concentration in all figures is 200nM. All data contain the average of three biological replicates, unless otherwise stated. Error bars are &#xB1;standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Proteasome dysfunction causes accumulation of the tau cleavage fragment.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Treatment with proteasome inhibitor MG132 at 1&#xB5;M decreases levels of the 25kD rotenone-induced tau fragment. &lt;strong&gt;(B)&lt;/strong&gt; Treatment with rotenone decreases proteasome activity (top) but not levels (bottom). &lt;strong&gt;(C)&lt;/strong&gt; Quantitation of &lt;strong&gt;(B)&lt;/strong&gt;, average of six biological replicates. &lt;strong&gt;(D)&lt;/strong&gt; Knockdown of PSME2 increases fragment levels (left) Western blot with quantitation. Over-expression with CRISPRa of &lt;em&gt;PSME1&lt;/em&gt; and &lt;em&gt;PSME2&lt;/em&gt; decrease fragment levels (right). Full gels in Figure S6. &lt;strong&gt;(E)&lt;/strong&gt; Expression of GFP-tau&lt;sup&gt;Metless&lt;/sup&gt; decreases fragment formation as compared to GFP-tau in neurons. Western blot (left), quantitation of western blot (right). &lt;strong&gt;(F)&lt;/strong&gt; Traces of averaged intensity from peptides per amino acid derived from mass spectrometry data for full-length tau fragment (black) and rotenone induced fragment (dark red). A sharp decrease in intensity is seen in the rotenone induced fragment gel slice after residue 200, except for one peptide (see Figure S7). Stars represent neo-tryptic termini identified upon digestion with GluC, pinpointing two possible C-termini for the fragment, narrowing the fragment identity to residues 172-200 (2N4R numbering: 230-258). &lt;strong&gt;(G)&lt;/strong&gt; Western blot of cell lysate (top) and undiluted neuron-conditioned media (bottom) shows fragment in the media. Rotenone concentration in all figures is 200nM. All data contain the average of three biological replicates, unless otherwise stated. Error bars are &#xB1;standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F6.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F6.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <span class="caption-title">Proteasome dysfunction causes accumulation of the tau cleavage fragment.</span><p id="p-33" class="first-child"><strong>(A)</strong> Treatment with proteasome inhibitor MG132 at 1µM decreases levels of the 25kD rotenone-induced tau fragment. <strong>(B)</strong> Treatment with rotenone decreases proteasome activity (top) but not levels (bottom). <strong>(C)</strong> Quantitation of <strong>(B)</strong>, average of six biological replicates. <strong>(D)</strong> Knockdown of PSME2 increases fragment levels (left) Western blot with quantitation. Over-expression with CRISPRa of <em>PSME1</em> and <em>PSME2</em> decrease fragment levels (right). Full gels in <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure S6</a>. <strong>(E)</strong> Expression of GFP-tau<sup>Metless</sup> decreases fragment formation as compared to GFP-tau in neurons. Western blot (left), quantitation of western blot (right). <strong>(F)</strong> Traces of averaged intensity from peptides per amino acid derived from mass spectrometry data for full-length tau fragment (black) and rotenone induced fragment (dark red). A sharp decrease in intensity is seen in the rotenone induced fragment gel slice after residue 200, except for one peptide (see <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure S7</a>). Stars represent neo-tryptic termini identified upon digestion with GluC, pinpointing two possible C-termini for the fragment, narrowing the fragment identity to residues 172-200 (2N4R numbering: 230-258). <strong>(G)</strong> Western blot of cell lysate (top) and undiluted neuron-conditioned media (bottom) shows fragment in the media. Rotenone concentration in all figures is 200nM. All data contain the average of three biological replicates, unless otherwise stated. Error bars are ±standard deviation.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutation of Tau C291 does not affect fragment formation. (A) Western blot of neurons transduced with GFP-tau or GFP-tau C291A treated with rotenone or vehicle. (B). Quantitation of data in (A). All samples are the average of three biological replicates, error bars are &#xB1;standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mutation of Tau C291 does not affect fragment formation.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Western blot of neurons transduced with GFP-tau or GFP-tau C291A treated with rotenone or vehicle. &lt;strong&gt;(B).&lt;/strong&gt; Quantitation of data in (A). All samples are the average of three biological replicates, error bars are &#xB1;standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F11.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F11.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5</span> <span class="caption-title">Mutation of Tau C291 does not affect fragment formation.</span><p id="p-76" class="first-child"><strong>(A)</strong> Western blot of neurons transduced with GFP-tau or GFP-tau C291A treated with rotenone or vehicle. <strong>(B).</strong> Quantitation of data in (A). All samples are the average of three biological replicates, error bars are ±standard deviation.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gels quantified in Figure 6 (A) CRISPRa of PSME1.(B) CRISPRa of PSME2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Gels quantified in Figure 6&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; CRISPRa of PSME1.&lt;strong&gt;(B)&lt;/strong&gt; CRISPRa of PSME2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F12.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F12.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6</span> <span class="caption-title">Gels quantified in <a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6</a></span><p id="p-77" class="first-child"><strong>(A)</strong> CRISPRa of PSME1.<strong>(B)</strong> CRISPRa of PSME2.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots of peptides and their intensities from mass spectrometry experiments. (A) Rotenone induced fragment excised from SDS/PAGE gel. (B) Full length tau excised from SDS/PAGE gel (C) Purified tau peptides from hydrogen peroxide treated neurons digested with GluC. (D) Purified tau peptides from 200nM rotenone treated neurons digested with GluC. (E) Purified tau peptides from vehicle treated neurons digested with GluC. Stars denote neo-tryptic termini." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Plots of peptides and their intensities from mass spectrometry experiments.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Rotenone induced fragment excised from SDS/PAGE gel. &lt;strong&gt;(B)&lt;/strong&gt; Full length tau excised from SDS/PAGE gel &lt;strong&gt;(C)&lt;/strong&gt; Purified tau peptides from hydrogen peroxide treated neurons digested with GluC. &lt;strong&gt;(D)&lt;/strong&gt; Purified tau peptides from 200nM rotenone treated neurons digested with GluC. &lt;strong&gt;(E)&lt;/strong&gt; Purified tau peptides from vehicle treated neurons digested with GluC. Stars denote neo-tryptic termini.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F13.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F13.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7</span> <span class="caption-title">Plots of peptides and their intensities from mass spectrometry experiments.</span><p id="p-78" class="first-child"><strong>(A)</strong> Rotenone induced fragment excised from SDS/PAGE gel. <strong>(B)</strong> Full length tau excised from SDS/PAGE gel <strong>(C)</strong> Purified tau peptides from hydrogen peroxide treated neurons digested with GluC. <strong>(D)</strong> Purified tau peptides from 200nM rotenone treated neurons digested with GluC. <strong>(E)</strong> Purified tau peptides from vehicle treated neurons digested with GluC. Stars denote neo-tryptic termini.</p><div class="sb-div caption-clear"></div></div></div><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="qPCR-based quantification effect of sgRNAs used in this study on expression of the targeted genes. (A) sgRNAs targeting MAPT in CRISPRi. (B) sgRNAs targeting CUL5 in CRISPRi. (C) sgRNAs targeting RNF7 in CRISPRi. (D) sgRNAs targeting SOCS4 in CRISPRa. (E) sgRNAs targeting PSME2 in CRISPRi. (F) sgRNAs targeting PSME1 in CRISPRa. (G) sgRNAs targeting PSME2 in CRISPRa. All samples are the average of three biological replicates, error bars are &#xB1;standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379620760" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;qPCR-based quantification effect of sgRNAs used in this study on expression of the targeted genes.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; sgRNAs targeting &lt;em&gt;MAPT&lt;/em&gt; in CRISPRi. &lt;strong&gt;(B)&lt;/strong&gt; sgRNAs targeting &lt;em&gt;CUL5&lt;/em&gt; in CRISPRi. &lt;strong&gt;(C)&lt;/strong&gt; sgRNAs targeting &lt;em&gt;RNF7&lt;/em&gt; in CRISPRi. &lt;strong&gt;(D)&lt;/strong&gt; sgRNAs targeting &lt;em&gt;SOCS4&lt;/em&gt; in CRISPRa. &lt;strong&gt;(E)&lt;/strong&gt; sgRNAs targeting &lt;em&gt;PSME2&lt;/em&gt; in CRISPRi. &lt;strong&gt;(F)&lt;/strong&gt; sgRNAs targeting &lt;em&gt;PSME1&lt;/em&gt; in CRISPRa. &lt;strong&gt;(G)&lt;/strong&gt; sgRNAs targeting &lt;em&gt;PSME2&lt;/em&gt; in CRISPRa. All samples are the average of three biological replicates, error bars are &#xB1;standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F14.medium.gif" width="440" height="367"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8" src="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F14.medium.gif" width="440" height="367"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2023/06/26/2023.06.16.545386/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8</span> <span class="caption-title">qPCR-based quantification effect of sgRNAs used in this study on expression of the targeted genes.</span><p id="p-79" class="first-child"><strong>(A)</strong> sgRNAs targeting <em>MAPT</em> in CRISPRi. <strong>(B)</strong> sgRNAs targeting <em>CUL5</em> in CRISPRi. <strong>(C)</strong> sgRNAs targeting <em>RNF7</em> in CRISPRi. <strong>(D)</strong> sgRNAs targeting <em>SOCS4</em> in CRISPRa. <strong>(E)</strong> sgRNAs targeting <em>PSME2</em> in CRISPRi. <strong>(F)</strong> sgRNAs targeting <em>PSME1</em> in CRISPRa. <strong>(G)</strong> sgRNAs targeting <em>PSME2</em> in CRISPRa. All samples are the average of three biological replicates, error bars are ±standard deviation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Proteasome activity, measured by native gel, of neurons treated with rotenone revealed a fifty percent decrease in activity of 30S, 26S and 20S proteasomes (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B-C</a>), although no changes in levels of proteasomal proteins were observed. Thus, we hypothesize that changes in proteasome processivity, rather than levels, may promote formation of the proteolytic tau fragment.</p><p id="p-35">Upon induction of oxidative stress, the proteasome activator PA.28 is often up-regulated or increases its association with the 20S proteasome in order to deal with increased proteostatic load due to the oxidation of proteins<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. Overexpression by CRISPRa of the two PA.28 subunits, PA.28⍺ and PA.28β, encoded by the genes <em>PSME1</em> and <em>PSME2</em> respectively, decreased 25-kD fragment formation (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6D</a>, left). Knockdown of the PA.28 subunit PA.β increased fragment levels dramatically (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a>, right), further suggesting that it is the inability of the proteasome to processively degrade oxidized proteins that leads to fragment accumulation.</p><p id="p-36">We next hypothesized that direct tau oxidation of tau leads to fragment formation. 0N3R tau only has one cysteine, C291, which is located close to the microtubule-binding domain. Expression of a GFP-0N3R tau transgene with the cysteine mutated to alanine (C291A) did not change fragment levels compared to a transgene with the cysteine (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure S4</a>). Therefore, we hypothesized that oxidation of tau’s six methionines, 5 of which are in the first 75 residues of tau, could be directly oxidized upon induction of oxidative stress and thus lead to proteasome misprocessing. We cloned a methionine-free version of tau, tau<sup>Metless</sup>, where every methionine is mutated to leucine. This construct, although expressed at the same levels as WT tau, showed a large decrease in fragment formation (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6E</a>). Taken together, we hypothesize that direct oxidation of tau by ROS allows the proteasome to aberrantly engage with and proteolyze tau, leaving the observed 25kD N-terminal fragment.</p><p id="p-37">We then over-expressed GFP-Tau and purified the fragment for identification by mass spectrometry (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6F</a>). We observed a substantial decrease in the observance and intensity of peptides at approximately the 200th residue of the 0N3R tau sequence. However, considering that we did not identify any non-tryptic peptides in that region that would serve as the C-terminus for a fragment digested by an endogenous protease, we hypothesized that the digestion may be performed by a trypsin-like protease, consistent with trypsin-like activity of the proteasome. This led us to pursue our second strategy, an in-solution digest of GFP-tau purified from neurons that had been treated with rotenone using the protease GluC. After GluC digestion and LC-MS/MS, we identified three peptides that were semi-specific for GluC with tryptic N-termini, ending at 172 and 176 in the 0N3R MAPT sequence. Thus, the fragment sequence is narrowed to a small region spanning residues 172-200 in this region (2N4R tau residues 230-258). This is remarkably similar to the tau biomarker peptides present in tauopathy patient CSF, which end near residue 230, and tau fragments previously identified in iPSC-neuron conditioned media<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>–<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. ELISA-based CSF tauopathy biomarker assays that use an N-terminal antibody and phospho-tau epitopes 181 or 231 are the most accurate for disease prediction<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>. We then asked whether the 25kD-fragment could be secreted from neurons. Indeed, we were able to detect its presence in the conditioned media of neurons, which increased upon rotenone treatment (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6G</a>). Thus, we hypothesize that N-terminal tau biomarkers could be markers of neuronal oxidative stress and resulting changes in proteasome activity.</p></div></div><div class="section" id="sec-8"><h2 class="">Discussion</h2><p id="p-38">Here we established that <em>MAPT</em> V337M iPSC-derived neurons have higher levels of tau oligomers than <em>MAPT</em> WT iPSC-derived neurons. We performed a genome-wide CRISPRi-screen to uncover cellular factors controlling tau oligomer accumulation. To complement this primary we screen, we performed several small-scale CRISPRi screens using orthogonal tau oligomer specific antibodies and total tau antibodies in both the <em>MAPT</em> V337M and <em>MAPT</em> WT backgrounds. This strategy uncovered known modulators of tau and tau oligomer levels, validating this approach, and revealed novel regulators of tau and tau oligomer levels, such as <em>CUL5</em>.</p><p id="p-39">In comparison to other tau screens in the literature<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>–<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup>, our screen adds a disease-relevant context for tau genetic modifier screens: human iPSC-derived neurons. In combination with our use of conformational-specific antibodies, we are able to discriminate between those genes affecting only tau levels or only tau misfolding. We performed screens in both mutant (V337M) and wildtype tau genotypes and cross-compared these screens for factors specific to genotype and antibody used.</p><p id="p-40">Generally, pathways identified in our screens agree with those previously identified in the literature. Novel pathways elucidated in our screens, such as GPI-anchor proteins or UFMylation, may be neuron-specific, as the currently published large-scale screens were performed in cancer cell lines. A previous screen performed in SHY5Y cells<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>, identified <em>CUL5</em> knockdown as a negative modifier of tau levels, contrary to our work, which identifies <em>CUL5</em> knockdown as a positive modifier of tau levels. Our earlier finding that CUL5 is expressed more highly in human entorhinal cortex neurons resilient to Alzheimer’s disease than in those that are selectively vulnerable<sup><a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a></sup> highlights CUL5 as a potential causal determinant of selective vulnerability in human brains. Further work will need to be done to identify the mechanisms by which CUL5 regulates tau levels.</p><p id="p-41">Knockdown of nuclear-encoded mitochondrial genes is the primary signature of the genome-wide screen presented here, and knockdown of those genes increases tau oligomer levels. This signature adds to a growing body of literature that connects tau function, mitochondrial function, oxidative stress, and neurodegenerative disease<sup><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>–<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>. Mechanistic studies here reveal one possible mechanism by which CSF-based biomarkers for tauopathies may be generated. We found that ETC inhibition generates acute oxidative stress, which in turn leads to proteasome misprocessing of oxidized tau. The resultant protein fragments are very similar in sequence to known CSF biomarkers<sup><a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a>–<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">52</a></sup>. There are other possible mechanisms by which mitochondrial dysfunction may also relate to tau misfolding or dysfunction. For instance, knockdown of ETC complex I genes has recently been recognized to increase tolerance for cellular ROS. More mechanistic work must be done to fully elucidate the relationship between tau and mitochondrial biology.</p></div><div class="section" id="sec-9"><h2 class="">STAR★Methods</h2><div id="sec-10" class="subsection"><h3>KEY RESOURCES TABLE</h3><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/3229684/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/3229684/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div></div></div></div><div class="section" id="sec-11"><h2 class="">RESOURCE AVAILABLILITY</h2><div id="sec-12" class="subsection"><h3>LEAD CONTACT AND MATERIALS AVAILABILITY</h3><p id="p-42">The cell lines generated in this study are available on request upon the completion of a Material Transfer Agreement (MTA). All plasmids generated in this study will be deposited on AddGene. Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Martin Kampmann (<span class="em-link"><span class="em-addr">martin.kampmann{at}ucsf.edu</span></span>).</p></div><div id="sec-13" class="subsection"><h3>DATA AND CODE AVAILABILITY</h3><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-43">CRISPR screening data have been deposited at CRISPR-brain (<a href="https://www.crisprbrain.org/">https://www.crisprbrain.org/</a>) and will be publicly available as of the date of publication.</p></li><li id="list-item-2"><p id="p-44">CRISPR screening raw data will be made available upon request</p></li><li id="list-item-3"><p id="p-45">All original code is available at <a href="https://kampmannlab.ucsf.edu/scripts-tau-oligomer-screen-manuscript">https://kampmannlab.ucsf.edu/scripts-tau-oligomer-screen-manuscript</a>.</p></li></ul></div><div id="sec-14" class="subsection"><h3>EXPERIMENTAL MODEL AND SUBJECT DETAILS</h3><div id="sec-15" class="subsection"><h4>Human iPSCs</h4><p id="p-46">Human iPSCs (in the male WTC11 background (Miyaoka et al., 2014) were cultured in StemFlex Medium on BioLite Cell Culture Treated Dishes (Thermo Fisher Scientific; assorted Cat. No.) coated with Growth Factor Reduced, Phenol Red-Free, LDEV-Free Matrigel Basement Membrane Matrix (Corning; Cat. No. 356231) diluted 1:100 in Knockout DMEM (GIBCO/Thermo Fisher Scientific; Cat. No. 10829-018). Routine passaging was performed as described <sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>. Studies with human iPSCs at UCSF were approved by the The Human Gamete, Embryo and Stem Cell Research (GESCR) Committee. Informed consent was obtained from the human subjects when the WTC11 (Miyaokaet al., 2014) lines were originally derived.</p></div><div id="sec-16" class="subsection"><h4>Human iPSC-derived neurons</h4><p id="p-47">Human iPSC-derived neurons were pre-differentiated and differentiated as described <sup><a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></sup>. Briefly, iPSCs were pre-differentiated in Matrigel-coated plates or dishes in N2 Pre-Differentiation Medium containing the following: KnockOut DMEM/F12 as the base, 1× MEM non-essential amino acids, 1× N2 Supplement (Gibco/Thermo Fisher Scientific, cat. no. 17502-048), 10 ng ml<sup>−1</sup> of NT-3 (PeproTech, cat. no. 450-03), 10 ng ml<sup>−1</sup> of BDNF (PeproTech, cat. no. 450-02), 1 μg ml<sup>−1</sup> of mouse laminin (Thermo Fisher Scientific, cat. no. 23017-015), 10 nM ROCK inhibitor and 2 μg ml<sup>−1</sup> of doxycycline to induce expression of mNGN2. After 3 d, on the day referred to hereafter as Day 0, pre-differentiated cells were re-plated into BioCoat poly-D-lysine-coated plates or dishes (Corning, assorted cat. no.) in regular neuronal medium, which we refer to as +AO neuronal medium, containing the following: half DMEM/F12 (Gibco/Thermo Fisher Scientific, cat. no. 11320-033) and half neurobasal-A (Gibco/Thermo Fisher Scientific, cat. no. 10888-022) as the base, 1× MEM non-essential amino acids, 0.5× GlutaMAX Supplement (Gibco/Thermo Fisher Scientific, cat. no. 35050-061), 0.5× N2 Supplement, 0.5× B27 Supplement (Gibco/Thermo Fisher Scientific, cat. no. 17504-044), 10 ng ml<sup>−1</sup> of NT-3, 10 ng ml<sup>−1</sup> of BDNF and 1 μg ml<sup>−1</sup> of mouse laminin. Neuronal medium was half-replaced every week. Full protocols are available on protocols.io (<span class="underline">dx.doi.org/10.17504/protocols.io.bcrjiv4n</span>).</p></div><div id="sec-17" class="subsection"><h4>HEK293T</h4><p id="p-48">HEK293Ts were cultured in DMEM supplemented with 10%FBS, 1% Pen/Strep, 1% Glutamine (DMEM complete). Cells were passaged by washing with DPBS, adding trypsin for 2 minutes at 37C, and quenching and resuspended in a 5-fold trypsin volume of DMEM complete. Cells were spun at 200xg for 5 minutes, counted, and plated at the desired density.</p></div></div></div><div class="section" id="sec-18"><h2 class="">METHOD DETAILS</h2><div id="sec-19" class="subsection"><h3>Generation of V337M CRISPRi line</h3><p id="p-49">WTC11 WT/V337M iPSCs harboring a single-copy of doxycycline-inducible mouse NGN2 at the AAVS1 locus<sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup> were used as the parental iPSC line for further genetic engineering. iPSCs were transfected with pC13N-dCas9-BFP-KRAB and TALENS targeting the human CLYBL intragenic safe harbor locus (between exons 2 and 3) (pZT-C13-R1 and pZT-C13-L1, Addgene #62196, #62197) using DNA In-Stem (VitaScientific). After 14 days, BFP-positive iPSCs were isolated via FACS sorting, and individualized cells were plated in a serial dilution series to enable isolation of individual clones under direct visualization with an inverted microscope in a tissue culture hood via manual scraping. Clones with heterozygous integration of dCas9-BFP-KRAB (determined using PCR genotyping) were used for further testing. Full protocols are available on protocols.io: <a href="https://dx.doi.org/10.17504/protocols.io.8dahs2e">https://dx.doi.org/10.17504/protocols.io.8dahs2e</a>.</p></div><div id="sec-20" class="subsection"><h3>Flow cytometry</h3><div id="sec-21" class="subsection"><h4>Fixed neurons</h4><p id="p-50">For CRISPR-screening, iPSC-differentiated neurons were washed with HBSS and then dissociated from the plate using Papain solution (20 U/mL papain in HBSS) at 37C for 10 minutes. Papain was quenched with 3x volume DMEM with 10% FBS and spun down 200xg for 10 minutes. Cells were then fixed with zinc fixation buffer (0.1M Tris-HCl, pH 6.5 @ 4C, 0.5% ZnCl2, 0.5% Zn Acetate, 0.05% CaCl2) overnight at 4C. When ready for staining, samples were washed three times in TBS by centrifugation at 200xg for 10 minutes and resuspended in permeabilization buffer (10% Normal Goat Serum, 10% 10x TBS, 3% BSA, 1% Glycine, 0.5% Tween-20) and blocked for 30 minutes. During blocking, cells were triturated into a single-cell suspension with progressively smaller pipette tips, from P1000 pipette tips to P20. After addition of primary antibodies, the antibody/cell slurry was incubated for two hours. Samples were then spun down at 200xg for 10 minutes and washed 3x with TBS by centrifugation. The cell pellet was then resuspended in permeabilization buffer with secondary antibodies and incubated at room temperature for one hour. Samples were then spun down at 200xg for 10 minutes and washed 3x with TBS by centrifugation. In the last wash, Hoechst at a concentration of 2 µM was added to the wash buffer for 10 minutes at room temperature. After the last wash, cells were resuspended in 1 mL of TBS and FACS sorted on a BD ARIA Fusion.</p><p id="p-51">All other iPSC-differentiated neurons were prepared for flow cytometry in 96-well plate format as follows. iPSC-differentiated neurons were washed with TBS and then fixed with zinc fixation buffer at 4 °C overnight. When ready for staining, samples were washed three times in TBS and then 50uL of permeabilization buffer was added and incubated for 30 minutes to block. Primary antibody in permeabilization buffer was then added and samples incubated at 4 °C overnight. Primary was removed and cells were washed 4x in TBS. Secondary antibodies were added in permeabilization buffer and incubated for one hour at room temperature. Cells were washed 4x in TBS with Hoechst in the second to last wash. After the last wash, cells were triturated by pipetting up and down with a P200 tip 10 times and then a P20 tip 10 times and analyzed with a BD Celesta cell analyzer and the data were processed using FlowJo v10 software.</p></div><div id="sec-22" class="subsection"><h4>Live neurons</h4><p id="p-52">iPSC-differentiated neurons were washed with HBSS and dissociated from the plate using Papain solution (20 U/mL papain in HBSS) at 37 °C for 10 minutes. Papain was quenched with 3x volume DMEM with 10% FBS and spun down 200xg for 10 minutes. Cells were then resuspended in HBSS and analyzed with a BD Celesta analyzer the data were processed using FlowJo v10 software. For cells analyzed with CellRox (thermo), CellRox diluted to 5 µM in differentiation media was added 1:1 with the extant well media volume and incubated for 30 minutes at 37 °C. Cells were washed with HBSS three times and dissociated and flowed as described above.</p></div></div><div id="sec-23" class="subsection"><h3>Lentivirus generation</h3><div id="sec-24" class="subsection"><h4>sgRNAs</h4><p id="p-53">Lentivirus was generated as described <sup><a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a></sup>. Briefly, HEK293T were plated to achieve 80-95% confluence 24 hours after plating. For a 6-well plate, 1ug of transfer plasmid, and 1ug of third generation packaging mix, were diluted into 200 µL OPTIMEM and 12 µL of TRANSIT-LENTI was added. Transfection mix was incubated at room temperature for 10 minutes and then added to HEKs. After 48 hours, supernatant was transferred to a syringe and filtered through a 0.45 µm PVDF filter into a conical tube. ¼ volume of Lentivirus Precipitation Solution was added to the filtrate and stored at 4 °C for 24 hours. Lentivirus-containing supernatant was then centrifuged for 30 min at 1500xg at 4 °C, supernatant aspirated, and then spun again at 4 °C for 5 mins at 1500xg. Supernatant was aspirated and the virus-containing pellet was resuspended in 200 µL DPBS. For screening, each library was prepared from a 15cm plate of HEK293Ts, scaled appropriately. For infection, virus was added at the same time as iPSC passaging. After 48 hours, cells were passaged, analyzed for marker positivity by flow cytometry and selected with 1 µg/mL puromycin until &gt;95% marker positive, for two passages. Cells were allowed to recover for one passage before pre-differentiation. The full protocol is available on protocols.io (<a href="https://dx.doi.org/10.17504/protocols.io.8dfhs3n">https://dx.doi.org/10.17504/protocols.io.8dfhs3n</a>)</p></div><div id="sec-25" class="subsection"><h4>Over-expression constructs</h4><p id="p-54">Lentivirus was generated as described <sup><a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">23</a></sup>. Briefly, HEK293T were plated to achieve 80-95% confluence 24 hours after plating. For a 6-well plate, 1 µg of transfer plasmid, and 1 µg of third-generation packaging mix, were diluted into 200uL OPTIMEM and 12 µL of TRANSIT-LENTI was added. Transfection mix was incubated at room temperature for 10 minutes and then added to HEK293T cells. After 48 hours, supernatant was transferred to a syringe and filtered through a 0.45 µm PVDF filter into a conical tube. ¼ volume of Lentivirus Precipitation Solution was added to the filtrate and stored at 4 °C for 24 hours. Lentivirus-containing supernatant was then centrifuged for 30 min at 1500xg at 4 °C, supernatant aspirated, and then spun again at 4 °C for 5 mins at 1500xg. Supernatant was aspirated and the virus-containing pellet was resuspended in 200n µL DPBS. Cells were expanded to T75 flasks and when 80% confluent sorted for the appropriate marker (mApple or GFP).</p></div></div><div id="sec-26" class="subsection"><h3>CRISPRi screening</h3><p id="p-55">For each genome-wide sub-library and each secondary screen, 45 million iPSCs in 3x T175s were infected with lentivirus as above at an MOI of ∼0.3 and selected. Cells were then differentiated as above and plated on 3x 15-cm PDL-coated dishes at a density of 15 million cells per plate. Cells were then matured for two weeks and prepared for FACS sorting as above, staining for NeuN and the tau-specific antibodies as indicated. For the tau-specific antibodies with mouse host, NeuN staining was not performed. Cells were collected into 1mL of 30% BSA in a FACS tube. After sorting, cells were pelleted at 200xg for 20 minutes, the supernatant was removed and the pellet was frozen at −20. Genomic DNA was extracted with the NucleoSpin Blood L kit. sgRNA cassettes were amplified pooled and sequenced as described <sup><a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">23</a></sup>. Sequencing was analyzed as described for each sub-library <sup><a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">23</a></sup>. Screening data is available in Supplemental Tables 2 and 3.</p></div><div id="sec-27" class="subsection"><h3>Cloning of secondary screen library</h3><p id="p-56">A pool of sgRNA-containing oligonucleotides were synthesized by Agilent Technologies and cloned into our optimized sgRNA expression vector as previously described<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>.</p></div><div id="sec-28" class="subsection"><h3>Western Blotting</h3><p id="p-57">iPSC-derived neurons were cultured as described above. Neurons were washed 3x with ice-cold DPBS and then ice-cold RIPA with protease and phosphatase inhibitors was added to cells (50 µL for a 24-well plate). Lysates were incubated on ice for 2 minutes and then scraped down.</p><p id="p-58">Lysates were either flash frozen on liquid nitrogen or directly centrifuged at 21000xg for 10 minutes at 4 °C. The supernatants were then collected, and concentrations assessed using a BCA assay (Thermo). 10 µg protein were loaded onto a 4-12% Bis-Tris polyacrylamide gel (Thermo). Nitrocellulose membranes were used to transfer the protein in a BioRad Transblot for 11 minutes at 25 V, 25 A. Membranes were then blocked for 1 hour with Licor Odyssey block and primary was added in Licor Odyssey block overnight at 4 °C. Blots were then washed 4x 5 minutes with TBST and secondary antibodies were added in Licor Odyssey block for 1 hour at room temperature. Blots were washed 4x 5 minutes with TBST and imaged on a Licor. Immunoblots were quantified by intensity using ImageStudio (Licor).</p></div><div id="sec-29" class="subsection"><h3>Immunoprecipitations</h3><div id="sec-30" class="subsection"><h4>Flag IPs</h4><p id="p-59">iPSC-derived neurons were cultured as described above. Neurons were washed 3x with ice-cold DPBS and then lysed in FLAG-lysis buffer (20 mM HEPES NaOH pH 7.4, 150 mM NaCl, 0.2% NP40 with protease and phosphatase inhibitors and 2 µM 1,10-phenathroline). Lysates were freeze/thawed on liquid nitrogen 7 times and then centrifuged at 21000xg for 30 minutes at 4C. The supernatants were then collected and concentrations assessed using a BCA assay. 5 mg of lysate was loaded onto 25 µL FLAG dynabeads (thermo) that had been washed 3x with FLAG-lysis buffer. IP was performed with rotation at 4 °C for 2 hours. Beads were then washed 3x with FLAG-lysis buffer and then 2x with TBS. FLAG-peptide elution was performed with 1 mg/mL FLAG-peptide in TBS overnight at 4C. Acid elution with 100 mM glycine pH 2.0 was performed for 10 min and the supernatant was quenched in 1/10<sup>th</sup> volume of 1M Tris pH 9.0. High-temperature elutions were performed using 2x LDS with 20 mM DTT for 70 °C for 5 minutes.</p></div><div id="sec-31" class="subsection"><h4>GFP IPs</h4><p id="p-60">iPSC-derived neurons were cultured as described above. Neurons were washed 3x with ice-cold DPBS and then lysed in GFP-lysis buffer supplemented with protease and phosphatase inhibitors (ProteinTech). Lysates were freeze/thawed on liquid nitrogen 7 times and then centrifuged at 21000xg for 30 minutes at 4 °C. The supernatants were then collected and diluted using GFP dilution buffer supplemented with protease and phosphatase inhibitors (ProteinTech).</p><p id="p-61">Concentrations assessed using a BCA assay. 100 mg of lysate was loaded onto 100 µL GFP-trap beads (ProteinTech) that had been washed 3x with Dilution buffer supplemented with protease and phosphatase inhibitors. IP was performed with rotation at 4 °C for 1 hours. Beads were then washed 3x with GFP-wash buffer and then 2x with TBS. Acid elution with 100 mM glycine pH 2.0 was performed for 10 min and the supernatant was quenched in 1/10<sup>th</sup> volume of 1M Tris pH 9.0. Eluates were then loaded on a gel and excised for mass spectrometry (below).</p></div></div><div id="sec-32" class="subsection"><h3>In-gel proteasome assay</h3><p id="p-62">Proteasome activity from neurons was measured as described (<span class="underline">10.1016/j.xpro.2021.100526</span>). Briefly, iPSC-derived neurons were washed 3x with ice cold DPBS, and then scraped into TSDG lysis buffer (10 mM Tris, 1.1 mM MgCl2, 10 mM NaCl, 1 mM NaN3, 1 mM DTT, 2 mM ATP, 10% glycerol) and lysed by freeze/thaw on liquid nitrogen 7 times. Samples were then spun down 10 minutes at 21000xg, assayed by BCA, and 50 µg supernatant was mixed with 5x native gel loading buffer (0.05% bromophenol blue, 43.5% glycerol, 250 mM Tris pH 7.5) to a concentration of 1x loading buffer. Samples were loaded on a 3-8% Tris Acetate gel (Thermo) was run in native gel running buffer (1x TBE, 413 µM ATP, 2 mM MgCl2, 0.5 mM DTT) for 3 hours at 170 V. Gels were then incubator for 30 min at 37 °C in reaction buffer (50 mM Tris pH 7.5, 10 mM MgCl2, 1 mM ATP, 1 mM DTT, 48 µM Suc-LLVY-AMC) and imaged on a Gel Doc EZ (Bio-rad).</p></div><div id="sec-33" class="subsection"><h3>Mass spectrometry</h3><p id="p-63">Gel bands were then excised and the proteins making up each band were reduced, alkylated, and digested with trypsin all within the gel. The resulting peptides were then extracted from the gel slice into solution and analyzed using liquid chromatography tandem mass spectrometry (LC-MS/MS) with data-dependent acquisition. Preparation in solution was used because alternate proteases, such as GluC, cannot penetrate the gel matrix making gel separation and preparation infeasible. After acquiring and analyzing our LC-MS/MS data, we identified three peptides that were semi-specific for the GluC with tryptic N termini, ending at 172 and 176 in the fetal MAPT sequence.</p></div><div id="sec-34" class="subsection"><h3>qPCR</h3><p id="p-64">RNA was extracted using the Zymo Quick-RNA miniprep kit and cDNA was synthesized with the SensiFAST cDNA synthesis kit. Samples were prepared for qpCR in technical triplicates using SensiFAST SYBR Lo-ROX 2x Mastermix. qPCR was performed on an Applied Biosystems Quantstudio 6 Pro Real-Time PCR System using Quantstudio Real Time PCR Software following Fast 2-Step protocol: (1) 95 °C for 20 s; (2) 95 °C for 5 s (denaturation); (3) 60 °C for 20 s (annealing/extension); (4) repeat steps 2 and 3 for a total of 40 cycles; (5) 95 °C for 1 s; (6) ramp 1.92 °C s<sup>−1</sup> from 60 °C to 95 °C to establish melting curve. Expression fold changes were calculated using the ΔΔCt method, normalizing to housekeeping gene <em>GAPDH</em>. Primer sequences are provided in Table S1. Quantification of qPCR samples are shown in <a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure S7</a>.</p></div><div id="sec-35" class="subsection"><h3>Drug treatments</h3><p id="p-65">At the two-week feeding, 50% of the media volume was removed and drugs diluted in media were added in order to obtain the correct drug concentration when adding media to reach the previous media volume.</p></div></div><div class="section" id="sec-36"><h2 class="">QUANTIFICATION AND STATISTICAL ANALYSIS</h2><p id="p-66">For statistical analysis, we used GraphPad Prism 9.5.1. Data are shown as mean±SD, except for flow cytometry data, which is shown as the median±SD. For two sample comparions, un unpaired two-tailed Student’s t-test was used. For three sample comparison, a two-way ANOVA was used. P-values are shown above compared samples, n.s. denotes not significant.</p><div id="sec-37" class="subsection"><h3>CRISPR-screen analysis</h3><div id="sec-38" class="subsection"><h4>Primary screen analysis</h4><p id="p-67">CRISPR screens were analyzed using MAGeCK-iNC as previously described <sup><a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">23</a></sup>. Briefly, raw sequencing reads were cropped and aligned using custom scripts that are already publicly available (<a href="https://kampmannlab.ucsf.edu/resources">https://kampmannlab.ucsf.edu/resources</a>). Raw phenotype scores and p-values were calculated for target genes and negative control genes using a Mann-Whitney U-test. Hit genes were identified using a FDR of 0.05. Gene scores were normalized by the standard deviation of negative controls genes for each genome-wide sublibrary. Hits were then combined and gene set enrichment analysis (GSEA) was performed for T22 positive and negative bins after filtering for mitochondrial genes using ENRICHR<sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>–<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>.</p></div><div id="sec-39" class="subsection"><h4>Pairwise analysis secondary screens</h4><p id="p-68">After data processing as described above, normalized hits files for re-test screens were compared using custom python scripts to calculate Pearson’s correlation coefficient and generate lists of genes that were unique to each screen in the comparison using a gene score of ±5. We then performed GSEA using ENRICHR on these unique gene sets in order to label categories of genes. Venn diagrams of overlapping hits were generated using a custom python script and the list of overlapping genes processed for GSEA using ENRICHR, after filtering for mitochondrial genes.</p></div></div></div><div class="section" id="sec-41"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-70">A.J.S. and M.K. conceived the project. A.J.S., N.A., J.M., G.R., E.C., C.P.B., S.G., D.G., C.C., D.S. designed and performed experiments. A.J.S., N.A., G.R., E.C., C.P.B., J.M., R.A., D.E.,P.G., R.T., N.M.K., J.W., D.S., L.G., M.K. developed experimental protocols, tools, and reagents or analyzed data. A.J.S. and M.K. wrote the manuscript.</p></div><div class="section" id="sec-42"><h2 class="">DECLARATION OF INTERESTS</h2><p id="p-71">M. K. is an inventor on US Patent 11,254,933 related to CRISPRi and CRISPRa screening, serves on the Scientific Advisory Boards of Engine Biosciences, Casma Therapeutics, Cajal Neuroscience, Alector, and Montara Therapeutics, and is an advisor to Modulo Bio and Recursion Therapeutics. The other authors declare no competing interests.</p></div><div class="section" id="sec-44"><h2 class="">Supplemental Table Legends</h2><p id="p-80"><strong>Table S1 – qPCR primers and sgRNA protospacer sequences used in this study.</strong></p><p id="p-81"><strong>Table S2 – Screen results for primary and secondary screens. Each screen is in a separate tab.</strong> First column is gene. Second column is phenotype (log2 fold change of high bin over low bin). Third column is p-value. Fourth column is gene score (phenotype times -log10(p-value)). See methods for determination of phenotype and p-value.</p><p id="p-82"><strong>Table S3 – All hit genes for primary and secondary screens. Each screen is in a separate tab.</strong> First column is gene. Second column is phenotype (log2 fold change of high bin over low bin). Third column is p-value. Fourth column is gene score (phenotype times -log10(p-value)). See methods for determination of phenotype and p-value.</p><p id="p-83"><strong>Table S4 – sgRNA protospacers sequences for secondary screen library</strong></p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-69">We would like to thank Dr Rakez Kayed for tau-oligomer specific antibodies. We would like to acknowledge E. Chow (UCSF Center for Advanced Technology) for support with next-generation sequencing; S. Elmes (UCSF Laboratory for Cell Analysis) for support with FACS, Grant P30CA082103. We would like to thank all members of the Kampmann lab for helpful advice and technical support. A.J.S was supported by NIH F32 AG063487 and NIH K99 AG080116-01. M.K. was supported by the Rainwater Charitable Foundation/Tau consortium, the Chan Zuckerberg Initiative Ben Barres Early Career Acceleration Award, and NIH grants R01 AG062359, R01 AG082141, U54 NS100717, U54 NS123746. D.E. and R.A. were supported by NIH U24AG072458 and HHMI.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> Department of Medical Neuroscience, School of Medicine, Southern University of Science and Technology, Shenzhen, Guangdong Province, China</p></li><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-5">Minor text edits and updated reference</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.1" data-doi="10.1126/science.abb8255"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">C.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shao</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mucke</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Tau: Enabler of diverse brain disorders and target of rapidly evolving therapeutic strategies</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.abb8255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChang%26rft.auinit1%253DC.-W.%26rft.volume%253D371%26rft.issue%253D6532%26rft.spage%253Deabb8255%26rft.epage%253Deabb8255%26rft.atitle%253DTau%253A%2BEnabler%2Bof%2Bdiverse%2Bbrain%2Bdisorders%2Band%2Btarget%2Bof%2Brapidly%2Bevolving%2Btherapeutic%2Bstrategies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb8255%26rft_id%253Dinfo%253Apmid%252F33632820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNzEvNjUzMi9lYWJiODI1NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIzLzA2LzI2LzIwMjMuMDYuMTYuNTQ1Mzg2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.2" data-doi="10.1038/nrc.2016.112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mandelkow</span>,  <span class="cit-name-given-names">E</span></span>. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Tau in physiology and pathology</span>. <abbr class="cit-jnl-abbrev">Nature reviews. Neuroscience</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">5</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrn.2015.1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BNeuroscience%26rft.volume%253D17%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc.2016.112%26rft_id%253Dinfo%253Apmid%252F26631930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc.2016.112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26631930&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.3" data-doi="10.1016/j.conb.2018.04.027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tracy</span>,  <span class="cit-name-given-names">T.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gan</span>,  <span class="cit-name-given-names">L</span></span>. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Tau-mediated synaptic and neuronal dysfunction in neurodegenerative disease</span>. <abbr class="cit-jnl-abbrev">Curr Opin Neurobiol</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">134</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.conb.2018.04.027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BNeurobiol%26rft.volume%253D51%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2018.04.027%26rft_id%253Dinfo%253Apmid%252F29753269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.conb.2018.04.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29753269&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.4" data-doi="10.1007/s00401-017-1707-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noble</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hanger</span>,  <span class="cit-name-given-names">D.P</span></span>. (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Roles of tau protein in health and disease</span>. <abbr class="cit-jnl-abbrev">Acta Neuropathol</abbr> <span class="cit-vol">133</span>, <span class="cit-fpage">665</span>–<span class="cit-lpage">704</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00401-017-1707-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BNeuropathol%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00401-017-1707-9%26rft_id%253Dinfo%253Apmid%252F28386764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00401-017-1707-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28386764&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.5" data-doi="10.1146/annurev.neuro.24.1.1121"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">V.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goedert</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Trojanowski</span>,  <span class="cit-name-given-names">J.Q</span></span>. (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Neurodegenerative tauopathies</span>. <abbr class="cit-jnl-abbrev">Annu Rev Neurosci</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1121</span>–<span class="cit-lpage">1159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.neuro.24.1.1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bneuroscience%26rft.stitle%253DAnnu%2BRev%2BNeurosci%26rft.aulast%253DLee%26rft.auinit1%253DV.%2BM-Y%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D1121%26rft.epage%253D1159%26rft.atitle%253DNEURODEGENERATIVE%2BTAUOPATHIES%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.neuro.24.1.1121%26rft_id%253Dinfo%253Apmid%252F11520930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.neuro.24.1.1121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11520930&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170109100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.6" data-doi="10.1038/31508"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hutton</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lendon</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rizzu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Froelich</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Houlden</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickering-Brown</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chakraverty</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Isaacs</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grover</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">Association of missense and 5’-splice-site mutations in tau with the inherited dementia FTDP-17</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">393</span>, <span class="cit-fpage">702</span>–<span class="cit-lpage">705</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/31508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHutton%26rft.auinit1%253DM.%26rft.volume%253D393%26rft.issue%253D6686%26rft.spage%253D702%26rft.epage%253D705%26rft.atitle%253DAssociation%2Bof%2Bmissense%2Band%2B5%2527-splice-site%2Bmutations%2Bin%2Btau%2Bwith%2Bthe%2Binherited%2Bdementia%2BFTDP-17.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F31508%26rft_id%253Dinfo%253Apmid%252F9641683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/31508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9641683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074289600058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.7" data-doi="10.1001/archneur.58.3.383"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poorkaj</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grossman</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steinbart</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Payami</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sadovnick</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nochlin</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tabira</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trojanowski</span>,  <span class="cit-name-given-names">J.Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borson</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galasko</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Frequency of tau gene mutations in familial and sporadic cases of non-Alzheimer dementia</span>. <abbr class="cit-jnl-abbrev">Arch Neurol</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">383</span>–<span class="cit-lpage">387</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1001/archneur.58.3.383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BNeurology%26rft.stitle%253DArch%2BNeurol%26rft.aulast%253DPoorkaj%26rft.auinit1%253DP.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D383%26rft.epage%253D387%26rft.atitle%253DFrequency%2Bof%2BTau%2BGene%2BMutations%2Bin%2BFamilial%2Band%2BSporadic%2BCases%2Bof%2BNon-Alzheimer%2BDementia%26rft_id%253Dinfo%253Adoi%252F10.1001%252Farchneur.58.3.383%26rft_id%253Dinfo%253Apmid%252F11255441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/archneur.58.3.383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11255441&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167441900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.8" data-doi="10.1111/j.1750-3639.2007.00052.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Swieten</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Spillantini</span>,  <span class="cit-name-given-names">M.G.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Hereditary frontotemporal dementia caused by Tau gene mutations</span>. <abbr class="cit-jnl-abbrev">Brain Pathol</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1750-3639.2007.00052.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrain%2Bpathology%2B%2528Zurich%252C%2BSwitzerland%2529%26rft.stitle%253DBrain%2BPathol%26rft.aulast%253Dvan%2BSwieten%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D73%26rft.atitle%253DHereditary%2Bfrontotemporal%2Bdementia%2Bcaused%2Bby%2BTau%2Bgene%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1750-3639.2007.00052.x%26rft_id%253Dinfo%253Apmid%252F17493040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1750-3639.2007.00052.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17493040&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244418400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.9" data-doi="10.1007/s12031-011-9593-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goedert</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Spillantini</span>,  <span class="cit-name-given-names">M.G</span></span>. (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Pathogenesis of the tauopathies</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Neuroscience</abbr> <span class="cit-vol">45</span>, <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12031-011-9593-4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bneuroscience%2B%253A%2B%2BMN%26rft.stitle%253DJ%2BMol%2BNeurosci%26rft.aulast%253DGoedert%26rft.auinit1%253DM.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DPathogenesis%2Bof%2Bthe%2Btauopathies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12031-011-9593-4%26rft_id%253Dinfo%253Apmid%252F21785996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12031-011-9593-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21785996&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.10" data-doi="10.1007/BF00308809"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Braak</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Braak</span>,  <span class="cit-name-given-names">E</span></span>. (<span class="cit-pub-date">1991</span>). <span class="cit-article-title">Neuropathological stageing of Alzheimer-related changes</span>. <abbr class="cit-jnl-abbrev">Acta Neuropathol</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">239</span>–<span class="cit-lpage">259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00308809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2Bneuropathologica%26rft.stitle%253DActa%2BNeuropathol%2B%2528Berl%2529%26rft.aulast%253DBraak%26rft.auinit1%253DH.%26rft.volume%253D82%26rft.issue%253D4%26rft.spage%253D239%26rft.epage%253D259%26rft.atitle%253DNeuropathological%2Bstageing%2Bof%2BAlzheimer-related%2Bchanges.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00308809%26rft_id%253Dinfo%253Apmid%252F1759558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00308809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1759558&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GG48700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.11" data-doi="10.1038/s41593-020-00764-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leng</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eser</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piergies</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sit</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neff</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>,  <span class="cit-name-given-names">R.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suemoto</span>,  <span class="cit-name-given-names">C.K.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Molecular characterization of selectively vulnerable neurons in Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">Nat Neurosci</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">276</span>–<span class="cit-lpage">287</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41593-020-00764-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BNeurosci%26rft.volume%253D24%26rft.spage%253D276%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-020-00764-7%26rft_id%253Dinfo%253Apmid%252F33432193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41593-020-00764-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=33432193&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.12" data-doi="10.1016/j.neuron.2016.01.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schö</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lockhart</span>,  <span class="cit-name-given-names">S.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schonhaut</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwimmer</span>,  <span class="cit-name-given-names">H.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabinovici</span>,  <span class="cit-name-given-names">G.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Correspondence</span>,  <span class="cit-name-given-names">W.J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schö Ll</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O</span> <span class="cit-name-surname">’neil</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.P.</span>, <span class="cit-name-surname">Janabi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span>, <span class="cit-name-surname">Ossenkoppele</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span>, <span class="cit-name-surname">et al</span></span>. (<span class="cit-pub-date">2016</span>). <abbr class="cit-jnl-abbrev">PET Imaging of Tau Deposition in the Aging Human Brain</abbr>. <span class="cit-fpage">971</span>–<span class="cit-lpage">982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2016.01.028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPET%2BImaging%2Bof%2BTau%2BDeposition%2Bin%2Bthe%2BAging%2BHuman%2BBrain%26rft.volume%253D89%26rft.spage%253D971%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2016.01.028%26rft_id%253Dinfo%253Apmid%252F26938442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2016.01.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26938442&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.13" data-doi="10.1016/j.neuron.2009.03.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seeley</span>,  <span class="cit-name-given-names">W.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crawford</span>,  <span class="cit-name-given-names">R.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">B.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Greicius</span>,  <span class="cit-name-given-names">M.D</span></span>. (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Neurodegenerative diseases target large-scale human brain networks</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">62</span>, <span class="cit-fpage">42</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2009.03.024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNeuron%26rft.aulast%253DSeeley%26rft.auinit1%253DW.%2BW.%26rft.volume%253D62%26rft.issue%253D1%26rft.spage%253D42%26rft.epage%253D52%26rft.atitle%253DNeurodegenerative%2Bdiseases%2Btarget%2Blarge-scale%2Bhuman%2Bbrain%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.03.024%26rft_id%253Dinfo%253Apmid%252F19376066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.03.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19376066&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265346700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.14" data-doi="10.1038/s41586-021-03911-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murzin</span>,  <span class="cit-name-given-names">A.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falcon</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kotecha</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Beers</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tarutani</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kametani</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garringer</span>,  <span class="cit-name-given-names">H.J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Structure-based classification of tauopathies</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">598</span>, <span class="cit-fpage">359</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41586-021-03911-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-021-03911-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-021-03911-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.15" data-doi="10.1016/j.cell.2020.01.027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arakhamia</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlomagno</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seyfried</span>,  <span class="cit-name-given-names">N.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrucelli</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">A.W.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arakhamia</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlomagno</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duong</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Posttranslational Modifications Mediate the Structural Diversity of Tauopathy Strains</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">180</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2020.01.027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.01.027%26rft_id%253Dinfo%253Apmid%252F32032505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.01.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32032505&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.16" data-doi="10.1016/j.cell.2020.10.029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wesseling</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mair</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlaffner</span>,  <span class="cit-name-given-names">C.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beerepoot</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fatou</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guise</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takeda</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Tau PTM Profiles Identify Patient Heterogeneity and Stages of Alzheimer’s Disease</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">183</span>, <span class="cit-fpage">1699</span>–<span class="cit-lpage">1713</span>.e13.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2020.10.029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D183%26rft.spage%253D1699%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.10.029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.10.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.17" data-doi="10.1038/s41588-022-01024-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bellenguez</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Küçükali</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">I.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kleineidam</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moreno-Grau</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amin</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Naj</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campos-Martin</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grenier-Boley</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andrade</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2022</span>). <span class="cit-article-title">New insights into the genetic etiology of Alzheimer’s disease and related dementias</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">54</span>, <span class="cit-fpage">412</span>–<span class="cit-lpage">436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-022-01024-z</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-022-01024-z%26rft_id%253Dinfo%253Apmid%252F35379992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-022-01024-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=35379992&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.18" data-doi="10.1038/s41588-021-00921-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wightman</span>,  <span class="cit-name-given-names">D.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">I.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savage</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shadrin</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bahrami</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rongve</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Børte</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Winsvold</span>,  <span class="cit-name-given-names">B.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drange</span>,  <span class="cit-name-given-names">O.K.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">A genome-wide association study with 1,126,563 individuals identifies new risk loci for Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">53</span>, <span class="cit-fpage">1276</span>–<span class="cit-lpage">1282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-021-00921-z</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D53%26rft.spage%253D1276%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-021-00921-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-021-00921-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.19" data-doi="10.1038/s41588-018-0311-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">I.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savage</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryois</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steinberg</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sealock</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karlsson</span>,  <span class="cit-name-given-names">I.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hägg</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Athanasiu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer’s disease risk</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">404</span>–<span class="cit-lpage">413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0311-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0311-9%26rft_id%253Dinfo%253Apmid%252F30617256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0311-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30617256&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.20" data-doi="10.1016/S1474-4422(19)30435-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fulton-Howard</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goate</span>,  <span class="cit-name-given-names">A</span></span>. (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Interpretation of risk loci from genome-wide association studies of Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">Lancet Neurol</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">326</span>–<span class="cit-lpage">335</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1474-4422(19)30435-1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BNeurol%26rft.volume%253D19%26rft.spage%253D326%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1474-4422%252819%252930435-1%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1474-4422(19)30435-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.21" data-doi="10.1038/s41586-019-1195-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mathys</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davila-Velderrain</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">J.Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menon</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abdurrob</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Single-cell transcriptomic analysis of Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">570</span>, <span class="cit-fpage">332</span>–<span class="cit-lpage">337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41586-019-1195-2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1195-2%26rft_id%253Dinfo%253Apmid%252F31042697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-019-1195-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31042697&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.22" data-doi="10.1016/j.neuron.2022.06.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Otero-Garcia</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mahajani</span>,  <span class="cit-name-given-names">S.U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wakhloo</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">Y.-Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morabito</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberhauser</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madira</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shakouri</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2022</span>). <span class="cit-article-title">Molecular signatures underlying neurofibrillary tangle susceptibility in Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">110</span>, <span class="cit-fpage">2929</span>–<span class="cit-lpage">2948</span>.e8.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2022.06.021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D110%26rft.spage%253D2929%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2022.06.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2022.06.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.23" data-doi="10.1016/j.neuron.2019.07.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gachechiladze</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ludwig</span>,  <span class="cit-name-given-names">C.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laurie</span>,  <span class="cit-name-given-names">M.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">J.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nathaniel</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prabhu</span>,  <span class="cit-name-given-names">A.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandopulle</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patel</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abshari</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">CRISPR Interference-Based Platform for Multimodal Genetic Screens in Human iPSC-Derived Neurons</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">104</span>, <span class="cit-fpage">239</span>–<span class="cit-lpage">255</span>.e12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2019.07.014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2019.07.014%26rft_id%253Dinfo%253Apmid%252F31422865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2019.07.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31422865&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.24" data-doi="10.1126/science.282.5395.1914"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">Mutation-Specific Functional Impairments in Distinct Tau Isoforms of Hereditary FTDP-17</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>, <span class="cit-fpage">1914</span>–<span class="cit-lpage">1917</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.282.5395.1914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHong%26rft.auinit1%253DM.%26rft.volume%253D282%26rft.issue%253D5395%26rft.spage%253D1914%26rft.epage%253D1917%26rft.atitle%253DMutation-Specific%2BFunctional%2BImpairments%2Bin%2BDistinct%2BTau%2BIsoforms%2Bof%2BHereditary%2BFTDP-17%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5395.1914%26rft_id%253Dinfo%253Apmid%252F9836646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5NS8xOTE0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjMvMDYvMjYvMjAyMy4wNi4xNi41NDUzODYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.25" data-doi="10.1212/WNL.0000000000003636"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spina</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schonhaut</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boeve</span>,  <span class="cit-name-given-names">B.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seeley</span>,  <span class="cit-name-given-names">W.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ossenkoppele</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Neil</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lazaris</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosen</span>,  <span class="cit-name-given-names">H.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boxer</span>,  <span class="cit-name-given-names">A.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perry</span>,  <span class="cit-name-given-names">D.C.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Frontotemporal dementia with the V337M MAPT mutation: Tau-PET and pathology correlations</span>. <abbr class="cit-jnl-abbrev">Neurology</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">758</span>– <span class="cit-lpage">766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1212/WNL.0000000000003636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurology%26rft.volume%253D88%26rft.spage%253D758%26rft_id%253Dinfo%253Adoi%252F10.1212%252FWNL.0000000000003636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1212/WNL.0000000000003636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.26" data-doi="10.1006/nbdi.2000.0335"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">C.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">I.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoon</span>,  <span class="cit-name-given-names">W.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryu</span>,  <span class="cit-name-given-names">B.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jo</span>,  <span class="cit-name-given-names">D.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">H.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwon</span>,  <span class="cit-name-given-names">Y.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gwag</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Proapoptotic effects of tau cleavage product generated by caspase-3</span>. <abbr class="cit-jnl-abbrev">Neurobiol Dis</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">162</span>–<span class="cit-lpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/nbdi.2000.0335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurobiology%2Bof%2Bdisease%26rft.stitle%253DNeurobiol%2BDis%26rft.aulast%253DChung%26rft.auinit1%253DC.%2BW.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D162%26rft.epage%253D172%26rft.atitle%253DProapoptotic%2Beffects%2Bof%2Btau%2Bcleavage%2Bproduct%2Bgenerated%2Bby%2Bcaspase-3.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fnbdi.2000.0335%26rft_id%253Dinfo%253Apmid%252F11162250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/nbdi.2000.0335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11162250&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167069500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.27" data-doi="10.1074/jbc.M112.396176"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flach</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilbrich</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schiffmann</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gärtner</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krüger</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leonhardt</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waschipky</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wick</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arendt</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Holzer</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Tau oligomers impair artificial membrane integrity and cellular viability</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">287</span>, <span class="cit-fpage">43223</span>–<span class="cit-lpage">43233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M112.396176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.396176%26rft_id%253Dinfo%253Apmid%252F23129775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvNTIvNDMyMjMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMy8wNi8yNi8yMDIzLjA2LjE2LjU0NTM4Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.28" data-doi="10.1021/bi1016233"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">G.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasagna-reeves</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castillo-carranza</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guerrero-mu</span>,  <span class="cit-name-given-names">M.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kayed</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2010</span>). <abbr class="cit-jnl-abbrev">Preparation and Characterization of Neurotoxic Tau Oligomers †</abbr>. <span class="cit-fpage">10039</span>–<span class="cit-lpage">10041</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/bi1016233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DLasagna-Reeves%26rft.auinit1%253DC.%2BA.%26rft.volume%253D49%26rft.issue%253D47%26rft.spage%253D10039%26rft.epage%253D10041%26rft.atitle%253DPreparation%2Band%2Bcharacterization%2Bof%2Bneurotoxic%2Btau%2Boligomers.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi1016233%26rft_id%253Dinfo%253Apmid%252F21047142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi1016233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21047142&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284438800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.29" data-doi="10.1096/fj.11-199851"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasagna-Reeves</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castillo-Carranza</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sengupta</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarmiento</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Troncoso</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">G.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kayed</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Identification of oligomers at early stages of tau aggregation in Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">The FASEB Journal</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">1946</span>–<span class="cit-lpage">1959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1096/fj.11-199851</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BFASEB%2BJournal%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.11-199851%26rft_id%253Dinfo%253Apmid%252F22253473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1096/fj.11-199851&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22253473&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303680800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasagna-Reeves</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castillo-Carranza</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sengupta</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guerrero-Munoz</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiritoshi</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neugebauer</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">G.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kayed</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Alzheimer brain-derived tau oligomers propagate pathology from endogenous tau</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr> <span class="cit-vol">2</span>. <span class="cit-fpage">10.1038/srep00700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D2%26rft.spage%253D101038srep00700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.31" data-doi="10.7554/eLife.12248"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Horlbeck</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villalta</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adamson</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pak</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fields</span>,  <span class="cit-name-given-names">A.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">C.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corn</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kampmann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Compact and highly active next-generation libraries for CRISPR-mediated gene repression and activation</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.19760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.12248%26rft_id%253Dinfo%253Apmid%252F26830461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.12248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26830461&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.32" data-doi="10.1038/s41467-021-22501-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caballero</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bourdenx</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luengo</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diaz</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sohn</span>,  <span class="cit-name-given-names">P.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juste</span>,  <span class="cit-name-given-names">Y.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wegmann</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patel</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Acetylated tau inhibits chaperone-mediated autophagy and promotes tau pathology propagation in mice</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">2238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-021-22501-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D12%26rft.spage%253D2238%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-021-22501-9%26rft_id%253Dinfo%253Apmid%252F33854069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-021-22501-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=33854069&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.33" data-doi="10.1016/j.jmb.2019.12.035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Djajadikerta</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keshri</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prestil</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryan</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rubinsztein</span>,  <span class="cit-name-given-names">D.C</span></span>. (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Autophagy Induction as a Therapeutic Strategy for Neurodegenerative Diseases</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">432</span>, <span class="cit-fpage">2799</span>–<span class="cit-lpage">2821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2019.12.035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BBiol%26rft.volume%253D432%26rft.spage%253D2799%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2019.12.035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2019.12.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.34" data-doi="10.1016/j.neuron.2017.01.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menzies</span>,  <span class="cit-name-given-names">F.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleming</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caricasole</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bento</span>,  <span class="cit-name-given-names">C.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ashkenazi</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Füllgrabe</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jimenez Sanchez</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karabiyik</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Autophagy and Neurodegeneration: Pathogenic Mechanisms and Therapeutic Opportunities</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">93</span>, <span class="cit-fpage">1015</span>–<span class="cit-lpage">1034</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2017.01.022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D93%26rft.spage%253D1015%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2017.01.022%26rft_id%253Dinfo%253Apmid%252F28279350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2017.01.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28279350&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.35" data-doi="10.1038/s41467-020-16984-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silva</span>,  <span class="cit-name-given-names">M.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nandi</span>,  <span class="cit-name-given-names">G.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tentarelli</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gurrell</span>,  <span class="cit-name-given-names">I.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jamier</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lucente</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dickerson</span>,  <span class="cit-name-given-names">B.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">D.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brandon</span>,  <span class="cit-name-given-names">N.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Haggarty</span>,  <span class="cit-name-given-names">S.J</span></span>. (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Prolonged tau clearance and stress vulnerability rescue by pharmacological activation of autophagy in tauopathy neurons</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">3258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-020-16984-1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D11%26rft.spage%253D3258%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-020-16984-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-020-16984-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.36" data-doi="10.1016/j.molcel.2021.07.038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ash</span>,  <span class="cit-name-given-names">P.E.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verma</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Vliet</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cruz</span>,  <span class="cit-name-given-names">A.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boudeau</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Interaction of tau with HNRNPA2B1 and N6-methyladenosine RNA mediates the progression of tauopathy</span>. <abbr class="cit-jnl-abbrev">Molecular Cell</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">4209</span>–<span class="cit-lpage">4227</span>.e12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2021.07.038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2021.07.038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2021.07.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.37" data-doi="10.1101/2023.06.19.544278"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Celeste Parra</span> <span class="cit-name-surname">Bravo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alice Maria</span> <span class="cit-name-surname">Giani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jesus</span> <span class="cit-name-surname">Madero-Perez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zeping</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Avi J</span> <span class="cit-name-surname">Samelson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Man Ying</span> <span class="cit-name-surname">Wong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Evangelisti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Li</span> <span class="cit-name-surname">Fan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Tatyana</span> <span class="cit-name-surname">Pozner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Maria</span> <span class="cit-name-surname">Mercedes</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2023</span>). <span class="cit-article-title">Human iPSC 4R tauopathy model uncovers modifiers of tau propagation</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, 2023.06.19.544278.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/2023.06.19.544278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2023.06.19.544278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMy4wNi4xOS41NDQyNzh2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIzLzA2LzI2LzIwMjMuMDYuMTYuNTQ1Mzg2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.38" data-doi="10.1093/jnen/nlv001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kanaan</span>,  <span class="cit-name-given-names">N.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez</span>,  <span class="cit-name-given-names">V.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stein</span>,  <span class="cit-name-given-names">T.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poncil</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McKee</span>,  <span class="cit-name-given-names">A.C</span></span>. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Characterization of Early Pathological Tau Conformations and Phosphorylation in Chronic Traumatic Encephalopathy</span>. <abbr class="cit-jnl-abbrev">J Neuropathol Exp Neurol</abbr> <span class="cit-vol">75</span>, <span class="cit-fpage">19</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jnen/nlv001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BNeuropathol%2BExp%2BNeurol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjnen%252Fnlv001%26rft_id%253Dinfo%253Apmid%252F26671985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jnen/nlv001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26671985&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.39" data-doi="10.1074/jbc.RA120.013638"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abskharon</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seidler</span>,  <span class="cit-name-given-names">P.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawaya</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cascio</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">T.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Philipp</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">C.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newell</span>,  <span class="cit-name-given-names">K.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghetti</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeTure</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Crystal structure of a conformational antibody that binds tau oligomers and inhibits pathological seeding by extracts from donors with Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">The Journal of biological chemistry</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.RA120.013638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bbiological%2Bchemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.RA120.013638%26rft_id%253Dinfo%253Apmid%252F32493775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTUvMzEvMTA2NjIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMy8wNi8yNi8yMDIzLjA2LjE2LjU0NTM4Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.40" data-doi="10.1021/acsomega.1c06890"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bano</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soomro</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abbas</span>,  <span class="cit-name-given-names">S.Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahmadi</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hassan</span>,  <span class="cit-name-given-names">S.S.U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Behl</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bungau</span>,  <span class="cit-name-given-names">S</span></span>. (<span class="cit-pub-date">2022</span>). <span class="cit-article-title">A Comprehensive Review of Biological Roles and Interactions of Cullin-5 Protein</span>. <abbr class="cit-jnl-abbrev">ACS Omega</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">5615</span>–<span class="cit-lpage">5624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/acsomega.1c06890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BOmega%26rft.volume%253D7%26rft.spage%253D5615%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facsomega.1c06890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acsomega.1c06890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">C</span></span>. (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">The functions and properties of cullin-5, a potential therapeutic target for cancers</span>. <abbr class="cit-jnl-abbrev">Am J Transl Res</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">618</span>– <span class="cit-lpage">632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BTransl%2BRes%26rft.volume%253D12%26rft.spage%253D618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.42" data-doi="10.1126/science.1089591"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">X.-F</span></span>. (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Induction of APOBEC3G ubiquitination and degradation by an HIV-1 Vif-Cul5-SCF complex</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>, <span class="cit-fpage">1056</span>–<span class="cit-lpage">1060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1089591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1089591%26rft_id%253Dinfo%253Apmid%252F14564014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDIvNTY0Ny8xMDU2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjMvMDYvMjYvMjAyMy4wNi4xNi41NDUzODYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.43" data-doi="10.1038/s41593-021-00862-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abarientos</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hashemi</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dräger</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leng</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nalls</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singleton</span>,  <span class="cit-name-given-names">A.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Genome-wide CRISPRi/a screens in human neurons link lysosomal failure to ferroptosis</span>. <abbr class="cit-jnl-abbrev">Nature Neuroscience</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1020</span>–<span class="cit-lpage">1034</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41593-021-00862-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D24%26rft.spage%253D1020%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-021-00862-0%26rft_id%253Dinfo%253Apmid%252F34031600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41593-021-00862-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=34031600&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.44" data-doi="10.1046/j.1471-4159.2002.01137.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">D.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Layfield</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serpell</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Narain</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goedert</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Spillantini</span>,  <span class="cit-name-given-names">M.G</span></span>. (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Proteasomal degradation of tau protein</span>. <abbr class="cit-jnl-abbrev">J Neurochem</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">176</span>–<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1471-4159.2002.01137.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurochemistry%26rft.stitle%253DJ%2BNeurochem%26rft.aulast%253DDavid%26rft.auinit1%253DD.%2BC.%26rft.volume%253D83%26rft.issue%253D1%26rft.spage%253D176%26rft.epage%253D185%26rft.atitle%253DProteasomal%2Bdegradation%2Bof%2Btau%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-4159.2002.01137.x%26rft_id%253Dinfo%253Apmid%252F12358741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-4159.2002.01137.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12358741&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178090600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.45" data-doi="10.3233/JAD-170959"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quinn</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">N.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellett</span>,  <span class="cit-name-given-names">K.A.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hooper</span>,  <span class="cit-name-given-names">N.M</span></span>. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Tau Proteolysis in the Pathogenesis of Tauopathies: Neurotoxic Fragments and Novel Biomarkers</span>. <abbr class="cit-jnl-abbrev">Journal of Alzheimer’s disease : JAD</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">13</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3233/JAD-170959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAlzheimer%253Fs%2Bdisease%2B%253A%2BJAD%26rft.volume%253D63%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.3233%252FJAD-170959%26rft_id%253Dinfo%253Apmid%252F29630551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3233/JAD-170959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29630551&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.46" data-doi="10.1186/s13024-023-00621-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sampognaro</span>,  <span class="cit-name-given-names">P.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arya</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knudsen</span>,  <span class="cit-name-given-names">G.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunderson</span>,  <span class="cit-name-given-names">E.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandoval-Perez</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodul</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowles</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Craik</span>,  <span class="cit-name-given-names">C.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">M.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kao</span>,  <span class="cit-name-given-names">A.W</span></span>. (<span class="cit-pub-date">2023</span>). <span class="cit-article-title">Mutations in α-synuclein, TDP-43 and tau prolong protein half-life through diminished degradation by lysosomal proteases</span>. <abbr class="cit-jnl-abbrev">Mol Neurodegener</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13024-023-00621-8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BNeurodegener%26rft.volume%253D18%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13024-023-00621-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13024-023-00621-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.47" data-doi="10.1126/sciadv.aay6350"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ukmar-Godec</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ibáñez</span> <span class="cit-name-surname">de Opakua</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span>, <span class="cit-name-surname">Henneberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span>, <span class="cit-name-surname">Godec</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span>, <span class="cit-name-surname">Pan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.T.</span>, <span class="cit-name-surname">Cima-Omori</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.S.</span>, <span class="cit-name-surname">Chari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span>, <span class="cit-name-surname">Mandelkow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span>, <span class="cit-name-surname">Urlaub</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span>, <span class="cit-name-surname">et al</span></span>. (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Proteasomal degradation of the intrinsically disordered protein tau at single-residue resolution</span>. <abbr class="cit-jnl-abbrev">Science Advances</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/sciadv.aba3916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aay6350%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/sciadv.aay6350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.48" data-doi="10.3389/fmolb.2019.00048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kors</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geijtenbeek</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reits</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schipper-Krom</span>,  <span class="cit-name-given-names">S</span></span>. (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Regulation of Proteasome Activity by (Post-)transcriptional Mechanisms</span>. <abbr class="cit-jnl-abbrev">Front Mol Biosci</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmolb.2019.00048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BMol%2BBiosci%26rft.volume%253D6%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmolb.2019.00048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmolb.2019.00048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.49" data-doi="10.1016/j.tcb.2004.11.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rechsteiner</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">C.P</span></span>. (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Mobilizing the proteolytic machine: cell biological roles of proteasome activators and inhibitors</span>. <abbr class="cit-jnl-abbrev">Trends Cell Biol</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">27</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tcb.2004.11.003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DRechsteiner%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D33%26rft.atitle%253DMobilizing%2Bthe%2Bproteolytic%2Bmachine%253A%2Bcell%2Bbiological%2Broles%2Bof%2Bproteasome%2Bactivators%2Band%2Binhibitors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2004.11.003%26rft_id%253Dinfo%253Apmid%252F15653075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2004.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15653075&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226765100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.50" data-doi="10.3233/JAD-150962"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barthélemy</span>,  <span class="cit-name-given-names">N.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabelle</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hirtz</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fenaille</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sergeant</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schraen-Maschke</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vialaret</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buée</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junot</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becher</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Differential Mass Spectrometry Profiles of Tau Protein in the Cerebrospinal Fluid of Patients with Alzheimer’s Disease, Progressive Supranuclear Palsy, and Dementia with Lewy Bodies</span>. <abbr class="cit-jnl-abbrev">J Alzheimers Dis</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">1033</span>–<span class="cit-lpage">1043</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3233/JAD-150962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAlzheimers%2BDis%26rft.volume%253D51%26rft.spage%253D1033%26rft_id%253Dinfo%253Adoi%252F10.3233%252FJAD-150962%26rft_id%253Dinfo%253Apmid%252F26923020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3233/JAD-150962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26923020&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.51" data-doi="10.1007/s00401-018-1948-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cicognola</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brinkmalm</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wahlgren</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Portelius</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gobom</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cullen</span>,  <span class="cit-name-given-names">N.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hansson</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parnetti</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Constantinescu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wildsmith</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Novel tau fragments in cerebrospinal fluid: relation to tangle pathology and cognitive decline in Alzheimer’s disease</span>. <abbr class="cit-jnl-abbrev">Acta Neuropathol</abbr> <span class="cit-vol">137</span>, <span class="cit-fpage">279</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00401-018-1948-2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BNeuropathol%26rft.volume%253D137%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00401-018-1948-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00401-018-1948-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.52" data-doi="10.1016/j.neuron.2018.02.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barthélemy</span>,  <span class="cit-name-given-names">N.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mawuenyega</span>,  <span class="cit-name-given-names">K.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">B.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gordon</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jockel-Balsarotti</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crisp</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kasten</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirmess</span>,  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Tau Kinetics in Neurons and the Human Central Nervous System</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">97</span>, <span class="cit-fpage">1284</span>–<span class="cit-lpage">1298</span>.e7.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2018.02.015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D97%26rft.spage%253D1284%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2018.02.015%26rft_id%253Dinfo%253Apmid%252F29566794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2018.02.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29566794&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.53" data-doi="10.1038/s41598-021-82658-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamm</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grinberg</span>,  <span class="cit-name-given-names">Y.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xi</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibilisco</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ravikumar</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautam</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Arrayed CRISPR reveals genetic regulators of tau aggregation, autophagy and mitochondria in Alzheimer’s disease model</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">2879</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-021-82658-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D11%26rft.spage%253D2879%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-021-82658-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-021-82658-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.54" data-doi="10.1016/j.neuron.2022.12.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Haro</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Ramahi</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garaicoechea</span>,  <span class="cit-name-given-names">L.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeong</span>,  <span class="cit-name-given-names">H.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sonn</span>,  <span class="cit-name-given-names">J.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tadros</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Botas</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zoghbi</span>,  <span class="cit-name-given-names">H.Y.</span></span> (<span class="cit-pub-date">2023</span>). <span class="cit-article-title">Evolutionarily conserved regulators of tau identify targets for new therapies</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">824</span>–<span class="cit-lpage">838</span>.e7.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2022.12.012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2022.12.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2022.12.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.55" data-doi="10.1038/s42003-021-02272-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>,  <span class="cit-name-given-names">C.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acker</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varadarajan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cochran</span>,  <span class="cit-name-given-names">N.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paula</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindeman</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McAllister</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Genome-wide CRISPR screen identifies protein pathways modulating tau protein levels in neurons</span>. <abbr class="cit-jnl-abbrev">Communications Biology</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s42003-021-02272-1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommunications%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs42003-021-02272-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s42003-021-02272-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.56" data-doi="10.1016/j.cell.2022.09.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaput</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shin</span>,  <span class="cit-name-given-names">M.-K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koh</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pieper</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">J.-A.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kang</span>,  <span class="cit-name-given-names">D.E</span></span>. (<span class="cit-pub-date">2022</span>). <span class="cit-article-title">X-linked ubiquitin-specific peptidase 11 increases tauopathy vulnerability in women</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">185</span>, <span class="cit-fpage">3913</span>–<span class="cit-lpage">3930</span>.e19.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2022.09.002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D185%26rft.spage%253D3913%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2022.09.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2022.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.57" data-doi="10.1186/1471-2202-11-144"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chaves</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melo</span>,  <span class="cit-name-given-names">T.Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martins</span>,  <span class="cit-name-given-names">S.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ferrari</span>,  <span class="cit-name-given-names">M.F.R</span></span>. (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Protein aggregation containing β-amyloid, α-synuclein and hyperphosphorylated τ in cultured cells of hippocampus, substantia nigra and locus coeruleus after rotenone exposure</span>. <abbr class="cit-jnl-abbrev">BMC Neurosci</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2202-11-144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bneuroscience%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BNeurosci%26rft.aulast%253DChaves%26rft.auinit1%253DR.%2BS.%26rft.volume%253D11%26rft.spage%253D144%26rft.epage%253D144%26rft.atitle%253DProtein%2Baggregation%2Bcontaining%2B%25CE%25B2-amyloid%252C%2B%25CE%25B1-synuclein%2Band%2Bhyperphosphorylated%2B%25CF%2584%2Bin%2Bcultured%2Bcells%2Bof%2Bhippocampus%252C%2Bsubstantia%2Bnigra%2Band%2Blocus%2Bcoeruleus%2Bafter%2Brotenone%2Bexposure.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2202-11-144%26rft_id%253Dinfo%253Apmid%252F21067569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2202-11-144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21067569&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.58" data-doi="10.1111/j.1471-4159.2005.03493.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Höglinger</span>,  <span class="cit-name-given-names">G.U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lannuzel</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khondiker</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michel</span>,  <span class="cit-name-given-names">P.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duyckaerts</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Féger</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Champy</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prigent</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Medja</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lombes</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">The mitochondrial complex I inhibitor rotenone triggers a cerebral tauopathy</span>. <abbr class="cit-jnl-abbrev">Journal of Neurochemistry</abbr> <span class="cit-vol">95</span>, <span class="cit-fpage">930</span>–<span class="cit-lpage">939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1471-4159.2005.03493.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurochemistry%26rft.stitle%253DJ%2BNeurochem%26rft.aulast%253DH%25C3%25B6glinger%26rft.auinit1%253DG.%2BU.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D930%26rft.epage%253D939%26rft.atitle%253DThe%2Bmitochondrial%2Bcomplex%2BI%2Binhibitor%2Brotenone%2Btriggers%2Ba%2Bcerebral%2Btauopathy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-4159.2005.03493.x%26rft_id%253Dinfo%253Apmid%252F16219024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-4159.2005.03493.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16219024&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232850000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.59" data-doi="10.1016/S0006-8993(02)03101-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chirichigno</span>,  <span class="cit-name-given-names">J.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manfredi</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beal</span>,  <span class="cit-name-given-names">M.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Albers</span>,  <span class="cit-name-given-names">D.S</span></span>. (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Stress-induced mitochondrial depolarization and oxidative damage in PSP cybrids</span>. <abbr class="cit-jnl-abbrev">Brain Res</abbr> <span class="cit-vol">951</span>, <span class="cit-fpage">31</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/s0006-8993(02)03101-3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrain%2Bresearch%26rft.stitle%253DBrain%2BRes%26rft.aulast%253DChirichigno%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D951%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D35%26rft.atitle%253DStress-induced%2Bmitochondrial%2Bdepolarization%2Band%2Boxidative%2Bdamage%2Bin%2BPSP%2Bcybrids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0006-8993%252802%252903101-3%26rft_id%253Dinfo%253Apmid%252F12231453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0006-8993(02)03101-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12231453&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178476900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.60" data-doi="10.1016/j.cell.2021.12.041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tracy</span>,  <span class="cit-name-given-names">T.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madero-Pérez</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swaney</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">T.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moritz</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konrad</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevenson</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hüttenhain</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kauwe</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2022</span>). <span class="cit-article-title">Tau interactome maps synaptic and mitochondrial processes associated with neurodegeneration</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2021.12.041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D185%26rft.spage%253D712%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2021.12.041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2021.12.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">61.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.61" data-doi="10.1016/j.neuron.2019.08.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sohn</span>,  <span class="cit-name-given-names">P.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosik</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sohn</span>,  <span class="cit-name-given-names">P.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <abbr class="cit-jnl-abbrev">Pathogenic Tau Impairs Axon Initial Segment Plasticity and Excitability Homeostasis Article Pathogenic Tau Impairs Axon Initial Segment Plasticity and Excitability Homeostasis</abbr>. <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2019.08.008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPathogenic%2BTau%2BImpairs%2BAxon%2BInitial%2BSegment%2BPlasticity%2Band%2BExcitability%2BHomeostasis%2BArticle%2BPathogenic%2BTau%2BImpairs%2BAxon%2BInitial%2BSegment%2BPlasticity%2Band%2BExcitability%2BHomeostasis%26rft.volume%253D104%26rft.spage%253D458%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2019.08.008%26rft_id%253Dinfo%253Apmid%252F31542321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2019.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31542321&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.62" data-doi="10.1016/j.stemcr.2017.08.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tracy</span>,  <span class="cit-name-given-names">T.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sohn</span>,  <span class="cit-name-given-names">P.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ludwig</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer-Franke</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Scalable Production of iPSC-Derived Human Neurons to Identify Tau-Lowering Compounds by High-Content Screening</span>. <abbr class="cit-jnl-abbrev">Stem Cell Reports</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1221</span>–<span class="cit-lpage">1233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.stemcr.2017.08.019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStem%2BCell%2BReports%26rft.volume%253D9%26rft.spage%253D1221%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stemcr.2017.08.019%26rft_id%253Dinfo%253Apmid%252F28966121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stemcr.2017.08.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28966121&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.63" data-doi="10.1016/j.cell.2014.09.029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horlbeck</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adamson</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villalta</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitehead</span>,  <span class="cit-name-given-names">E.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guimaraes</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panning</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ploegh</span>,  <span class="cit-name-given-names">H.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bassik</span>,  <span class="cit-name-given-names">M.C.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">159</span>, <span class="cit-fpage">647</span>–<span class="cit-lpage">661</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.09.029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D159%26rft.spage%253D647%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.09.029%26rft_id%253Dinfo%253Apmid%252F25307932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.09.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25307932&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000344521700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.64" data-doi="10.1186/1471-2105-14-128"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">E.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kou</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duan</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meirelles</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">N.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ma’ayan</span>,  <span class="cit-name-given-names">A</span></span>. (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-14-128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D128%26rft.atitle%253DENRICHR%253A%2BINTERACTIVE%2BAND%2BCOLLABORATIVE%2BHTML5%2BGENE%2BLIST%2BENRICHMENT%2BANALYSIS%2BTOOL%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-128%26rft_id%253Dinfo%253Apmid%252F23586463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23586463&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2023.06.16.545386v2.65" data-doi="10.1093/nar/gkw377"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuleshov</span>,  <span class="cit-name-given-names">M.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouillard</span>,  <span class="cit-name-given-names">A.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandez</span>,  <span class="cit-name-given-names">N.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duan</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koplev</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jagodnik</span>,  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lachmann</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Enrichr: a comprehensive gene set enrichment analysis web server 2016 update</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">W90</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw377%26rft_id%253Dinfo%253Apmid%252F27141961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27141961&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2023%2F06%2F26%2F2023.06.16.545386.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2023.06.16.545386v2.66" data-doi="10.1002/cpz1.90"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuleshov</span>,  <span class="cit-name-given-names">M.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clarke</span>,  <span class="cit-name-given-names">D.J.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evangelista</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lachmann</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wojciechowicz</span>,  <span class="cit-name-given-names">M.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kropiwnicki</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jagodnik</span>,  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2021</span>). <span class="cit-article-title">Gene Set Knowledge Discovery with Enrichr</span>. <abbr class="cit-jnl-abbrev">Curr Protoc</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">e90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/cpz1.90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BProtoc%26rft.volume%253D1%26rft.spage%253D90e%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcpz1.90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cpz1.90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span class="related-urls"></span></div></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2023.06.16.545386v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><span class="icon-chevron-up"></span> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2023.06.24.546342v2" title="Locomotor effects of a fibrosis-based immune response in stickleback fish" rel="prev" class="pager-link-prev link-icon"><span class="icon-circle-arrow-left"></span> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2021.11.15.468576v1" title="WITHDRAWN: In Silico Prediction and Comparison of Resistomes in Model Pseudomonas Strains by Resistance Gene Identifier (RGI)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <span class="icon-circle-arrow-right"></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;June 26, 2023.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2023.06.16.545386v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><span class="icon-external-link-sign"></span> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2023.06.16.545386v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2023.06.16.545386v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2023.06.16.545386v2" class="active" data-icon-position="" data-hide-link-title="0">CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/3222388" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-65UORsPGDztDXzEd7hOVg_n1XaJzpgkOsBcGYB6ybFU" />
<input type="hidden" name="form_id" value="forward_form" />
<fieldset class="captcha form-wrapper"><legend><span class="fieldset-legend">CAPTCHA</span></legend><div class="fieldset-wrapper"><div class="fieldset-description">This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.</div><input type="hidden" name="captcha_sid" value="676033385" />
<input type="hidden" name="captcha_token" value="f33f81b919370a6e820ed8ebc50beb37" />
<input type="hidden" name="captcha_response" value="Google no captcha" />
<div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image"></div></div></fieldset>
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><span class="icon-share-alt"></span> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article node3222388--3" data-node-nid="3222388" id="node-3222388--4517687741" data-pisa="biorxiv;2023.06.16.545386v2" data-pisa-master="biorxiv;2023.06.16.545386" data-seqnum="3222388" data-apath="/biorxiv/early/2023/06/26/2023.06.16.545386.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Avi J.</span> <span class="nlm-surname">Samelson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nabeela</span> <span class="nlm-surname">Ariqat</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">McKetney</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gita</span> <span class="nlm-surname">Rohanitazangi</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Celeste Parra</span> <span class="nlm-surname">Bravo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Darrin</span> <span class="nlm-surname">Goodness</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ruilin</span> <span class="nlm-surname">Tian</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Parker</span> <span class="nlm-surname">Grosjean</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Romany</span> <span class="nlm-surname">Abskharon</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">David</span> <span class="nlm-surname">Eisenberg</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Nicholas M.</span> <span class="nlm-surname">Kanaan</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Li</span> <span class="nlm-surname">Gan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Carlo</span> <span class="nlm-surname">Condello</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Danielle L.</span> <span class="nlm-surname">Swaney</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Kampmann</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2023.06.16.545386; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2023.06.16.545386 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2023.06.16.545386v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links text-center" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2&amp;title=CRISPR%20screens%20in%20iPSC-derived%20neurons%20reveal%20principles%20of%20tau%20proteostasis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2&amp;text=CRISPR%20screens%20in%20iPSC-derived%20neurons%20reveal%20principles%20of%20tau%20proteostasis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2&amp;t=CRISPR%20screens%20in%20iPSC-derived%20neurons%20reveal%20principles%20of%20tau%20proteostasis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.linkedin.com/shareArticle?mini=true&amp;url=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2&amp;title=CRISPR%20screens%20in%20iPSC-derived%20neurons%20reveal%20principles%20of%20tau%20proteostasis&amp;summary=&amp;source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2&amp;title=CRISPR%20screens%20in%20iPSC-derived%20neurons%20reveal%20principles%20of%20tau%20proteostasis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><span class="icon-globe"></span> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node3222388--5" data-node-nid="3222388" id="citation-node-3222388--62085829132" data-pisa="biorxiv;2023.06.16.545386v2" data-pisa-master="biorxiv;2023.06.16.545386" data-seqnum="3222388" data-apath="/biorxiv/early/2023/06/26/2023.06.16.545386.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">CRISPR screens in iPSC-derived neurons reveal principles of tau proteostasis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Avi J.</span> <span class="nlm-surname">Samelson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nabeela</span> <span class="nlm-surname">Ariqat</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">McKetney</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gita</span> <span class="nlm-surname">Rohanitazangi</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Celeste Parra</span> <span class="nlm-surname">Bravo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Darrin</span> <span class="nlm-surname">Goodness</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ruilin</span> <span class="nlm-surname">Tian</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Parker</span> <span class="nlm-surname">Grosjean</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Romany</span> <span class="nlm-surname">Abskharon</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">David</span> <span class="nlm-surname">Eisenberg</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Nicholas M.</span> <span class="nlm-surname">Kanaan</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Li</span> <span class="nlm-surname">Gan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Carlo</span> <span class="nlm-surname">Condello</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Danielle L.</span> <span class="nlm-surname">Swaney</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Kampmann</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2023.06.16.545386; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2023.06.16.545386 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=CRISPR%20screens%20in%20iPSC-derived%20neurons%20reveal%20principles%20of%20tau%20proteostasis&amp;rft.volume&amp;rft.issue&amp;rft.date=2023-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Samelson%2C+Avi+J.&amp;rft.au=Ariqat%2C+Nabeela&amp;rft.au=McKetney%2C+Justin&amp;rft.au=Rohanitazangi%2C+Gita&amp;rft.au=Bravo%2C+Celeste+Parra&amp;rft.au=Goodness%2C+Darrin&amp;rft.au=Tian%2C+Ruilin&amp;rft.au=Grosjean%2C+Parker&amp;rft.au=Abskharon%2C+Romany&amp;rft.au=Eisenberg%2C+David&amp;rft.au=Kanaan%2C+Nicholas+M.&amp;rft.au=Gan%2C+Li&amp;rft.au=Condello%2C+Carlo&amp;rft.au=Swaney%2C+Danielle+L.&amp;rft.au=Kampmann%2C+Martin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/3222388/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/3222388/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/3222388/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/3222388/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/3222388/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/3222388/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/3222388/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/3222388/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/3222388/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/3222388/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/3222388/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/3222388/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2&amp;count=horizontal&amp;via=&amp;text=CRISPR%20screens%20in%20iPSC-derived%20neurons%20reveal%20principles%20of%20tau%20proteostasis&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2023.06.16.545386v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2023.06.16.545386v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/cell-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Cell Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-new-inside">
<div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(5009)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(11226)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(8341)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(28131)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(14436)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(11558)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(16658)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(138)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(9076)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(13721)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2067)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(17767)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(11902)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(16291)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(11381)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(26947)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(11059)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(58524)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(432)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(1796)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(3091)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(4716)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(9992)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(1643)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(2777)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(7160)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(1570)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.js"></script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__mE-zA7XoPSPRAQoYjySuFvsmr_yQEp-d91PRLFbaefY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
  </body>
</html>
